\documentclass[11pt,twoside,lineno]{pnas-new} % was: twocolumn

% Add supporting info as file dependency so that we can reference the figures
\makeatletter
\newcommand*{\addFileDependency}[1]{% argument=file name and extension
  \typeout{(#1)}
  \@addtofilelist{#1}
  \IfFileExists{#1}{}{\typeout{No file #1.}}
}

\usepackage[backend=bibtex,style=nature]{biblatex} % Borrowing citation formatting from Nature, also not necessarily going there

\usepackage{xr}
\addFileDependency{manuscript/text_supporting.tex}%
\addFileDependency{manuscript/text_supporting.aux}%
\externaldocument{manuscript/text_supporting}%

\usepackage{setspace} % double-spacing

\bibliography{references}

\usepackage{xfp} % to do multiplication with \inteval

\input{manuscript/config_variables}
\input{manuscript/chain_summary_variables}
\input{manuscript/sample_summary_variables}
\input{manuscript/damping_factor_variables}
\input{manuscript/persistence}
\input{manuscript/intoductions_by_incidence_model}

\newcommand{\maxsamplingpercent}{\maxsamplingfraction*100}

\templatetype{pnasresearcharticle} % Borrowing template from PNAS, not necessarily going there 1st

\title{Genomic data enable quantification of the effect of  public health measures on reducing introductions and slowing local transmission in the 2020 Swiss SARS-CoV-2 epidemic}

% Use letters for affiliations, numbers to indicate the corresponding author
\author[a,b]{Sarah A. Nadeau}
\author[a,b]{Timothy G. Vaughan} 
\author[c]{Christiane Beckmann}
\author[a,b]{Ivan Topolsky}
\author[a,b]{Chaoran Chen}
\author[b,d]{Emma Hodcroft}
\author[a]{Tobias Schär}
\author[a]{Ina Nissen}
\author[a]{Natascha Santacroce}
\author[a]{Elodie Burcklen}
\author[a,b]{Pedro Ferreira}
\author[a,b]{Kim Philipp Jablonski}
\author[a,b]{Susana Posada-Céspedes}
\author[a]{Vincenzo Capece}
\author[a,b]{Sophie Seidel}
\author[e]{Noemi Santamaria de Souza}
\author[f]{Julia M. Martinez-Gomez}
\author[f]{Phil Cheng}
\author[f]{Philipp P. Bosshard}
\author[f]{Mitchell P. Levesque}
\author[g]{Verena Kufner}
\author[g]{Stefan Schmutz}
\author[g]{Maryam Zaheri}
\author[g]{Michael Huber}
\author[g]{Alexandra Trkola}
\author[h]{Samuel Cordey}
\author[h]{Florian Laubscher}
\author[i]{Ana Rita Gonçalves}
\author[tmp]{Sébastien Aeby}
\author[tmp]{Trestan Pillonel}
\author[tmp]{Damien Jacot}
\author[tmp]{Claire Bertelli}
\author[tmp]{Gilbert Greub}
\author[n,k]{Karoline Leuzinger}
\author[j,b,m]{Madlen Stange}
\author[j,b,m]{Alfredo Mari}
\author[j,b]{Tim Roloff}
\author[j,b]{Helena Seth-Smith}
\author[n]{Hans H. Hirsch}
\author[j,m]{Adrian Egli}
\author[c]{Maurice Redondo}
\author[c]{Olivier Kobel}
\author[c]{Christoph Noppen}
\author[a,b]{Niko Beerenwinkel}
\author[b,l]{Richard A. Neher}
\author[a]{Christian Beisel}
\author[a,b,1]{Tanja Stadler}

\affil[a]{Department of Biosystems Science and Engineering, ETH Zürich, Basel, Switzerland}
\affil[b]{Swiss Institute of Bioinformatics, Lausanne, Switzerland}
\affil[c]{Viollier AG, Allschwil, Switzerland}
\affil[d]{Institute for Social and Preventive Medicine, University of Bern, Bern, Switzerland}
\affil[e]{Department of Biology, ETH Zürich, Zürich, Switzerland}
\affil[f]{Department of Dermatology, University Hospital Zurich, University of Zürich, Zürich, Switzerland}
\affil[g]{Institute of Medical Virology, University of Zürich, Zürich, Switzerland}
\affil[h]{Laboratory of Virology, Division of Infectious Diseases and Division of Laboratory Medicine, University Hospitals of Geneva & Faculty of Medicine, University of Geneva, Geneva, Switzerland}
\affil[i]{Swiss National Reference Centre for Influenza, University Hospitals of Geneva, Geneva, Switzerland}
\affil[tmp]{Institute of Microbiology, University Hospital Centre and University of Lausanne, Lausanne, Switzerland}
\affil[j]{Clinical Bacteriology and Mycology, University Hospital Basel, Basel, Switzerland}
\affil[k]{Transplantation & Clinical Virology, Department Biomedicine, University of Basel, Basel, Switzerland}
\affil[l]{Biozentrum, University of Basel, Basel, Switzerland}
\affil[m]{Applied Microbiology Research, Department of Biomedicine, University of Basel, Basel, Switzerland}
\affil[n]{Clinical Virology, University Hospital Basel, Basel, Switzerland}

% Please include corresponding author, author contribution and author declaration information
% \authorcontributions{Please provide details of author contributions here.}
% \authordeclaration{Please declare any competing interests here.}
\correspondingauthor{\textsuperscript{1}To whom correspondence should be addressed. E-mail: tanja.stadler@bsse.ethz.ch}

% Three to five keywords, separated by the pipe symbol.
\keywords{Epidemiology $|$ SARS-CoV-2 $|$ Phylogenetics $|$ Phylodynamics} 

\begin{abstract}
% Please provide an abstract of no more than 250 words in a single paragraph. Abstracts should explain to the general reader the major contributions of the article. References in the abstract must be cited in full within the abstract itself and cited in the text.
Pathogen genome sequences are a unique source of information to trace routes of infection and track transmission dynamics through time. We sequenced 11,357 SARS-CoV-2 genomes in Switzerland throughout 2020, the 6th largest sequence data contribution globally in 2020. Here we use a representative subset of all Swiss sequences from 2020 to (i) identify putatively independent introductions into Switzerland and (ii) quantify within-Switzerland transmission dynamics. We confirm that in spring, SARS-CoV-2 was introduced many times prior to border closures but that most introductions were quickly driven to extinction during the spring lockdown. Introductions peaked before European incidence, indicating border closure reduced imports. In Summer 2020, introductions rose again as borders re-opened. With less stringent measures compared to spring, more of these introductions persisted over long periods in the local population. Finally, we estimate that in summer 2020, the transmission rate in newly introduced outbreaks was initially higher, then slowed down by \summermaxdampingpercentmedianCHEnosampUB\ - \summermindampingpercentmedianCHEnosampUB\%. A plausible explanation of this summertime ``slowdown'' dynamic is a successful test-trace-isolate implementation that roughly halved transmissions once an introduction was identified. This slowdown is not detected in fall which might point to contact tracing having been overburdened during the fall wave. Our analysis based on genomic sequencing data helps quantify the effects of international travel, general measures, and specific test-trace-isolate efforts towards a national epidemic, an increasingly relevant topic as global vaccination campaigns continue at an uneven pace and new variants of concern spread internationally.
\end{abstract}



\begin{document}

\maketitle
\abscontentformatted
\doublespacing

\section{Introduction}

% Claim importance
A crucial task in public health is to evaluate measures aimed at slowing SARS-CoV-2 transmission. Such measures aim to prevent new introductions into a region or contain ongoing transmission chains, for example with quarantine for travellers or social distancing. So far, mathematical models have helped estimate effects from broad measures like travel bans and lockdowns. These methods use data on case or death counts and travel volumes, e.g. \cite{Flaxman2020, Tian2020}. However, these data are not informative at the scale of individual introductions or transmission chains because there is no information to link related cases.

Detailed contact tracing data could help link cases to reconstruct introductions and transmission chains. However, these data are difficult to collect and not widely available. Instead, pathogen genome sequences can be used to link cases \cite{Kraemer}. Many RNA viruses accumulate mutations on the same time-scale as host-to-host transmission. For example a typical SARS-CoV-2 strain will accumulate $\sim$2 mutations per month along its $\sim$30,000 nucleotide-long genome \cite{Nextstrainteam}. These mutations allow us to construct a viral phylogeny (a ``family tree'' of viruses) that approximates the underlying host-to-host transmission tree. 

SARS-CoV-2 genomes were collected at an unprecedented scale in 2020 \cite{Alm2020} and they have been used extensively to characterize transmission dynamics. Two broad categories of methods are commonly used: phylogenetic and phylodynamic methods. Phylogenetic studies reconstruct a ``best-guess'' phylogeny and calculate statistics directly from this phylogeny without further explicit models. For example, \cite{Mallon2020, DuPlessis2021} showed that national lock-downs during the early Irish and English epidemics reduced lineage sizes and diversity according to reconstructed phylogenies. Phylodynamic studies assume the phylogeny must arise from some underlying model of transmission between hosts (and possibly migration of hosts between regions). This approach enables estimation of population-level transmission dynamics. For example, \cite{Miller2020, Geoghegan2020a, muller-wa-covid} showed that public health measures reduced SARS-CoV-2 transmission rates in Israel, New Zealand, and Washington State, USA. 

% Indicate a gap
It is rather difficult to quantify effects of public health measures using genomic data. Firstly, sequencing efforts vary by region and through time. Often, available samples are unrepresentative of the entire population \cite{Villabona-Arenas2020, DeMaio2015}. Secondly, SARS-CoV-2 sequencing was high compared to viral diversity, in particular during the early epidemic in spring 2020. If many sequences are similar or identical the phylogeny will contain polytomies. This means the phylogeny is only a coarse approximation of the  host-to-host transmission tree \cite{Villabona-Arenas2020}. Even worse, arbitrarily resolving polytomies may give misleading or overly confident estimates \cite{Morel2021}. Finally, real-world transmission dynamics are governed by complex population structure that is difficult to model. Highly parameterized phylodynamic models can help, as in \cite{Miller2020, Geoghegan2020a, Muller2020a}. Models with multiple compartments account for sampling biases and population structure and the phylogeny can be integrated out as a nuisance parameter. Recent developments are speeding these methods up \cite{Lemey2021}, but they are generally too computationally complex to fit to thousands of genomes.

% State that our paper fills the gap 
Here we present a two-step analysis framework carefully combining phylogenetic and phylodynamic methods to address these challenges and quantify transmission dynamics throughout 2020 in Switzerland. In each step, we condition on clusters in the sample set, an approach inspired by  \cite{Muller2020} and \cite{DuPlessis2021}. This ensures computational feasibility. To limit sampling biases, we down-sample sequences to be more spatio-temporally representative of the Swiss epidemic. Finally, we account for phylogenetic uncertainty from high sampling and comparatively low viral diversity by conservatively reporting estimates across a range of plausible polytomy resolutions.

% Emphasize our aims 
We use our framework to quantify transmission dynamics in Switzerland, with a particular focus on the effects of public health measures on the Swiss epidemic. First, we asked when introductions occurred and how long they persisted in Switzerland. We then asked how well introductions were controlled once they were identified by public health authorities. We reasoned that test-trace-isolate measures can only be enforced once authorities know about a case. Therefore, if successful, we hypothesize test-trace-isolate efforts will slow transmission after the first case in a transmission chain is identified. Although we focus on gaining these epidemiological insights from the Swiss epidemic, our framework can be applied to any regional SARS-CoV-2 epidemic with data available on GISAID \cite{GISAID}. We perform a small comparison using New Zealand data to illustrate this. 

\section{Results}
We analyzed \nfocalsamples\ SARS-CoV-2 genomes collected in Switzerland, representing the Swiss epidemic from the first detected case on 25. Feb. through \maxdate. These data are a sub-sample of all available Swiss genomes aimed to be more spatio-temporally representative of the Swiss epidemic compared to all available genomes while maintaining a large sample size (Figure  \ref{fig:downsampling_representativeness}). We limited the sampling period to avoid transmission rate variation due to more transmissible variants of concern, such as alpha, which arose in late 2020.

\subsection{Introductions, then persistence characterize the 2020 Swiss epidemic}

First, we identified independent introductions of SARS-CoV-2 into Switzerland using genome data and estimated their persistence. To do this, we divided Swiss SARS-CoV-2 genome sequences into different Pango lineages (Table S1) and constructed an approximate maximum-likelihood phylogeny for each lineage of Swiss and genetically similar foreign sequences. Next, we identified independently introduced singleton sequences or clusters of Swiss sequences from these lineage phylogenies as described in the Methods. Importantly, we identified two different sets of introductions resulting from two extreme resolutions of polytomies in the phylogenies. This allows us to report estimates across the range of strictly bifurcating phylogenies consistent with the genome data. We refer to these different sets of plausible introductions as ``few'' or ``many'' introductions.

We estimate the analyzed sequences originate from between \nchainsmin\ (few) and \nchainsmax\ (many) independent introductions into Switzerland. These introductions are roughly power law-distributed in size, with the 10 largest introductions accounting for \maxlargestchainsper\ \% to \minlargestchainsper \% of genomes (Figure \ref{fig:chain_size_dist}). From a down-sampling analysis, we see that we do not reach saturation - if we were to include more sequences, we would identify more introductions (Figure \ref{fig:sensitivity_figs}a). Therefore, the true number of introductions into Switzerland is difficult to estimate using our phylogenetic approach. 

However, since we sampled genome sequences proportional to the number of confirmed cases through time, we expect that trends through time in the number and persistence of introductions are representative of the underlying dynamics. Figure \ref{fig:chain-longevity}A shows that the number of new introductions each week rose exponentially from Feb. to Mar. 2020, coinciding with an exponential rise in European cases during this time (Figure \ref{fig:chain-longevity}B). Newly sampled introductions into Switzerland peaked the week of 15. Mar. (Figure \ref{fig:chain-longevity}A). Switzerland closed its borders 13. Mar. 2020 \cite{SWI-border-closure}. Given the delay between the true date of introduction and sampling, introductions likely peaked well before cases in Switzerland's neighboring countries and the rest of Europe (Figure \ref{fig:chain-longevity}B). After the border closure, newly sampled introductions dropped week-on-week until the week of 14. Jun. Switzerland re-opened its borders 15. Jun. 2020. Then, newly sampled introductions rose more-or-less steadily until Nov. 2020, coinciding with borders reopening across Europe and globally. 

Of course, new introductions cannot sustain a local epidemic unless they persist in the local population. Our analysis suggests several introductions were quite persistent in Switzerland, including \nspanningchainsfebnovmin\ that may have persisted from the start of the Swiss epidemic in Feb. 2020 until Nov. 2020 (Figure \ref{fig:chain-longevity-matrix}). On average, introductions persisted in Switzerland \meantimetolastsamplemax\ - \meantimetolastsamplemin\ days from first to last sampling. However, introductions in early 2020 were less persistent than later introductions. Figure \ref{fig:chain-longevity} shows the fraction of newly sampled introductions each month that persist at least 60 days. 0.5 - 8\% of Apr. introductions persisted this long, compared to 12 - 52\% of Sep. introductions. 

\begin{figure}[H]
\centering
\includegraphics[width=0.75\linewidth]{figures/introductions_and_persistence.png}
\caption{A. Introductions into Switzerland and their persistence by the week (top) or month(bottom) they were first sampled. Persistence is defined by $\geq$ 60 days between the first sample and the latest sample. The error bars span two point estimates generated assuming either few or many introductions. The shaded rectangles highlight when the Swiss border closure (13. Mar. - 15. Jun.) and partial lockdown (17. Mar. - 27. Apr.) happened. B. For comparison, weekly average incidence by European country/region around the time of the Swiss border closures on 13. Mar. 2020 (dashed line). Case data here comes from the ECDC \cite{ECDC}.}
\label{fig:chain-longevity}
\end{figure}

\subsection{Genome data capture summer 2020 ``slowdown'' transmission dynamics}

Next, we investigated epidemic control once SARS-CoV-2 variants were introduced in Switzerland. Here, we quantified time-varying transmission dynamics in Switzerland in a Bayesian phylodynamic framework. We used a birth-death model with serial sampling, as originally described by \cite{stadler_2010_bds}. We conditioned the model on the introductions identified from our phylogenetic analysis. In a nutshell, our model assumes that once lineages are introduced, they are transmitted between hosts according to a shared time-varying transmission rate, die out upon recovery/death of the host according to a constant becoming-uninfectious rate, and yield genome samples with a time-varying sampling probability. We assume individuals who test positive adhere to the self-isolation regulations, so lineages die out when they are sampled. Under this parameterization, the effective reproductive number in Switzerland is a function of the transmission rate, becoming-uninfectious rate, and sampling probability.

We extended this methodology by adding a transmission rate ``damping'' factor to the implementation as shown in Figure \ref{fig:phylo-methods}. The implementation of this new method is available at TODO. The method aims to test whether test-trace-isolate efforts in Switzerland slowed transmission once introductions were detected. We reasoned test-trace-isolate could slow transmission only shortly after the first case tests positive but not beforehand. We include this concept in our model by allowing the transmission rate to decrease by a multiplicative factor 2 days after each introduction is first sampled. The 2-day delay aims to account for the time between an individual giving a sample and contact tracers being notified. Finally, we allowed the damping factor to vary between spring, summer, and fall in Switzerland - periods characterized by very different case loads and testing regimes (Figure \ref{fig:scale-factor}a). We used a spike-and-slab prior on this factor to include the possibility of no transmission slowdown. 

\begin{figure}[h!]
\centering
\includegraphics[width=0.75\linewidth]{figures/phylodynamic_method_example.png}
\caption{Illustration of how transmission rate damping is modelled. (A) shows a background time-varying reproductive number Re before any damping. In each of the coloured areas (Green = Spring, orange = Summer, and purple = Fall), a different transmission rate dampening factor is assumed. The black boxes in (A) highlight the spread of two example introductions (B) and (C). The genome data sampled from these introductions are shown as red dots in (B) and (C). The appropriate damping factor on Re is applied to each introduction 2 days after the first genome sample (dashed line). More concretely, in each step of the MCMC (Markov chain Monte Carlo) analysis, the background Re (A), the three dampening factors, and the two phylogenies (B and C) are sampled. The likelihood of the sampled genome data is calculated given the ``applied'' effective reproductive number specific to each introduction (B \& C, bottom).}  
\label{fig:phylo-methods}
\end{figure}

Across several model configurations (see supplemental text) and our two polytomy assumptions, we estimate effective reproductive number values through time that roughly capture the same trends as estimates based on confirmed case counts (Figure \ref{fig:ReSampProbResults}). On the other hand, sampling probability estimates in fall 2020 are strongly dependent on the prior (Figure \ref{fig:ReSampProbResults}), which we discuss more in the supplemental text.

Based on our main model configuration, we estimate a \summermaxdampingpercentmedianCHEnosampUB\ to \summermindampingpercentmedianCHEnosampUB\% slowdown in transmission after introductions are first sampled during summer 2020. In comparison, we estimate no or very little slowdown in relation to first sampling during fall 2020 (Figure \ref{fig:scale-factor}). These results are qualitatively robust to conditioning on few or many introductions and imposing a strong prior bound on the sampling proportion (Figure \ref{fig:DampingFactorResults}). In contrast, estimates in spring 2020 are inconsistent. This could be because low genomic diversity in SARS-CoV-2 during this period causes high phylogenetic uncertainty. In summary, we report an evident ``slowdown'' dynamic in SARS-CoV-2 transmission in Switzerland in Summer 2020 where transmission slows when the first genome sample analyzed is collected.

\begin{figure}[h!]
\centering
\includegraphics[width=0.75\linewidth]{figures/contact_tracing_factors_no_sampUB_compared_to_cases.png}
\caption{Epidemic trajectories in (A) Switzerland and (B) New Zealand during 2020 shown as a 7-day rolling average of daily new confirmed cases \cite{Appel}. (C) and (D) show estimates for if and how much transmission rates were dampened after introductions were sampled during different time periods in (C) Switzerland and (D) New Zealand. The inference was done twice, once conditioning on introductions identified assuming many introductions (light grey) and once assuming few introductions (dark grey). Thus, the difference between estimates in light and dark grey are due to phylogenetic uncertainty.}  
\label{fig:scale-factor}
\end{figure}

\subsection{Comparison to the New Zealand epidemic}
While Switzerland is centrally located in Europe and well-connected to other countries, especially those in the (normally) barrier-free Schengen travel zone, New Zealand is a relatively isolated island nation. Additionally, New Zealand aimed to eradicate SARS-CoV-2 throughout 2020 using strong measures, such as keeping its borders closed \cite{ZL-covid-policies}, while Switzerland re-opened to Europe in early Summer 2020. We compared our estimates for the transmission damping factor between Switzerland in spring, summer, and fall 2020 with New Zealand before and after an epidemic breakpoint in mid-May when local transmission was briefly eradicated \cite{Geoghegan2020}. From this point until early August, all cases in New Zealand were linked to managed quarantine facilities at the border. Then, a new community outbreak was identified on 11 August \cite{Geoghegan2021-nzl-outbreak}. Case numbers were subsequently held at lower levels through Dec. 2020 (Figure \ref{fig:DampingFactorResults}B). Based on the available genome data from New Zealand, which includes samples linked to managed quarantine facilities and from the community, we estimated damping factors in New Zealand before and after 15. May to be comparable or stronger than in Switzerland during Summer and Fall 2020 (Figure \ref{fig:scale-factor}).

\section{Discussion}
% Re-state the goal and the main results
We aimed to assess genomic evidence for the impact of major public health measures on SARS-CoV-2 transmission in Switzerland. We estimate that the Swiss border closure on 16. Mar. 2020 caused new introductions of SARS-CoV-2 to peak before cases in Switzerland's neighboring countries and the rest of Europe. Without this intervention, we  expect new introductions would have continued rising with cases in neighboring regions until early to mid-April. The border closure was mostly lifted again on 15. Jun 2020. Correspondingly, we estimated week-on-week increases in the number of new introductions throughout summer 2020. Next, we estimated that introductions spreading during the spring 2020 lockdown were more likely to be driven to extinction compared to later introductions. Only 0.5 - 8\% of April introductions persisted more than 60 days, compared to 12 - 52\% of Septemper introductions. Finally, we quantified a summertime ``slowdown'' dynamic in which introductions initially spread faster, then slowed between \summermaxdampingpercentmedianCHEnosampUB\ and \summermindampingpercentmedianCHEnosampUB \%. Together, these results show the efficacy of border closure and lockdown, and potentially, quantify the efficacy of test-trace-isolate measures, in Switzerland.

% Contextualize the results: caveats
It is important to interpret these results in light of the data and methods used, and their limitations. Most obviously, estimates on the number, timing, and duration of introductions are very uncertain due to incomplete sampling and phylogenetic uncertainty. Others have struggled with the same challenges \cite{Morel2021} and we support their conclusion: one should consider estimates generated across a range of plausible phylogenies, as estimates based on any one phylogeny may be misleading. In our analysis, we present estimate ranges based on two extreme resolutions of polytomies. This is less computationally intensive, but more conservative, compared to generating a larger set of plausible phylogenies as in Bayesian analyses and as \cite{Morel2021} suggested for maximum-likelihood analyses.

Further, genomic data are observational data and subject to potential sampling biases. Available sequences may not represent the overall infected population due to time-varying testing regimes, geographic or demographic sampling biases, and demographic skew in disease severity. In particular, we expect severe cases, contacts of positive cases, and populations with better access to testing or better sequencing coverage are sampled more. To avoid some of these biases, we used our own sequences and other sequences collected across Switzerland and down-sampled the regions and time periods where sequences were over-represented compared to case numbers. Thus, we argue that our final data set is representative of trends in transmission dynamics through time. We caution against interpreting the absolute numbers of introductions identified and the persistence of any specific introduction. Finally, we expect changes through time to be primarily due to NPIs or seasonal effects because our sampling period is prior to the identification of more transmissible variants of concern and prior to vaccines being available. 

% Contextualize the results: interpretations
Despite these limitations, a major benefit of using whole-genome sequencing data is that we can link related cases. This allows us to identify independent introductions and to investigate transmission dynamics within introductions. Here, we hypothesized that test-trace-isolate measures should slow transmission within introductions once they are sampled. Without genomic data to differentiate cases into independent introductions and chains of transmission, this would only be possible if detailed contact tracing data were to become available. Recently, \cite{Fetzer2021} exploited an accidental, partial breakdown of English contact tracing to show that English contact tracing in early fall 2020 reduced transmissions in the 6 weeks following a positive case by 63\%. Here we use a different data source - whole-genome sequences - and, uniquely, track transmission dynamics within introductions. The measured efficacy of English contact tracing in early fall 2020 is within the range of our estimates for a transmission slowdown in Switzerland in summer 2020.

Although we hypothesize that a transmission ``slowdown'' should reflect test-trace-isolate measures, there could be other factors at play. First, extensive backwards and forwards contact tracing would have been necessary to simultaneously slow all sister lineages of the lineage first sampled, as we assume. Informal backwards contact tracing by individuals themselves may have helped, but likely not to the extent we assume. Instead, it is possible other factors helped the slowdown. For one, returning travellers have been implicated in transmitting more than non-travellers \cite{Hodcroft2021} during times of lifted restrictions as in summer 2020. Thus, a passive transmission slowdown might happen when the virus moves into the non-traveller population. However, we would expect travellers in fall to have similar contact networks as those in summer. Instead, transmission damping is only evident in Switzerland in summer 2020 when case counts were low. This could be because manual contact tracing (as in Switzerland) likely functions better when overall case loads are low. When case numbers were high in fall, Swiss contact tracing was reported to struggle \cite{SWI-contact-tracing-failing}. We also quantified a similar slowdown effect in New Zealand during two different time periods, indicating this effect is not unique to Switzerland during summer. \cite{Mendes2021} used genome sequence data to show that New Zealand contact tracing was highly effective in identifying SARS-CoV-2 infection clusters. Further, New Zealand is known for intense test-trace-isolate measures and quarantine for travellers \cite{ZL-covid-policies}. A second alternate explanation for the slowdown dynamic might be that contacts of positive cases are tested more intensely. For instance, close contacts are encouraged to get tested, yielding an apparent ``burst'' of samples around the first detected case. If so, we can still interpret the slowdown dynamic as evidence that test-trace-isolate measures were working, but it is difficult to say precisely how much transmission actually slowed.

% Argue for importance
Our findings highlight the effect of major public health measures on the Swiss epidemic. Our analysis suggests that strict public health measures like the partial border closure and lockdown in spring 2020 helped prevent and contain new introductions. However, these measures are naturally unsustainable. We estimate that when case counts are maintained at a low level, as in summer 2020 in Switzerland and in New Zealand, new introductions initially spread more quickly, but then slowed down. Estimates from fall 2020 in Switzerland show that no such slowdown occurred when case counts were high. We speculate this is due to test-trace-isolate resources being overwhelmed. In conclusion, our results argue for the benefits of test-trace-isolate efforts, thoughtful quarantine policies, and the benefits of maintaining case counts at a low level to enable outbreak control while vaccines are distributed and treatment options continue to improve.

\section{Methods}
\subsection*{Genome sequencing and quality filtering}
Most of the Swiss SARS-CoV-2 genomes analyzed until 31. Dec 2020 were generated by the Swiss SARS-CoV-2 Sequencing Consortium \cite{s3c-webpage}. Here, we briefly describe the swab-to-sequence process for these samples. RNA extracts from qPRC-positive patient naso- or oral-pharangeal swabs were provided by Viollier AG, a Swiss medical diagnostics company. RNA extraction was done with either the Abbott m2000sp or Seegene STARMag 96x4 Universal Cartridge RNA extraction kit. Extracts were then  transferred to the Genomics Facility Basel or the Functional Genomics Center Zurich for whole-genome sequencing. Both centers used the ARCTIC v3 primer scheme \cite{Quick2017, ARCTICNetwork} to generate tiled, approximately 400bp-long amplicons. Library preparation was done with the New England Biolabs (NEB) library preparation kit. Libraries were sequenced on Illumina MiSeq or NovaSeq machines, resulting in 2 x 251 base reads. Bioinformatics was done using V-pipe \cite{v-pipe-bioinformatics}, including read trimming and filtering with PRINSEQ \cite{Schmieder2011}, alignment to GenBank accession MN908947 \cite{Wu2020} with bwa \cite{Li2009}, and consensus base calling. For consensus base calling, positions with $<$5x coverage are masked with ``N'', positions with $>$5\% and $>$2 reads supporting a minor base are called with IUPAC ambiguity codes, and positions with $>$50\% reads supporting a deletion are called with ``-''. We rejected samples with $<$20,000 non-N bases. The consensus sequences we generated have been made publicly available on both GISAID (submitting lab: Department of Biosystems Science and Engineering, ETH Zürich) and European Nucleotide Archive (ENA) (study: PRJEB38472).

We supplemented our Swiss data with other Swiss sequences and foreign sequences available via GISAID (accessed 31. May 2021). From the full set of sequences available on GISAID, we removed from consideration non-human samples, samples $<$ 27000 bases long, and samples flagged by the Nextclade tool \cite{Aksamentov} for one of the following reasons: suspiciously clustered SNPs (QC SNP clusters status metric not ``good''; $>=$ 6 mutations in 100 bases), too many private mutations (QC private mutations status metric not ``good''; $>=$ 10 mutations from the nearest tree node), or overall bad quality (Nextclade QC overall status ``bad''). We aligned the sequences to the reference genome MN908947.3 using MAFFT \cite{katoh_mafft:_2002}. Finally, we followed the Nextstrain pipeline's recommendation to mask the first 100 and last 50 sites of the alignment \cite{Nextstraina} since the start and end of SARS-CoV-2 sequences are prone to sequencing error \cite{DeMaio2020}.

\subsection*{Sampling procedure}

From the quality-filtered alignment of GISAID sequences, we selected a focal set of sequences from Switzerland and a context set of sequences from abroad. 

% describe representativeness of Swiss dataset here too?
For the focal sequences from Switzerland, we aimed to select a spatially and temporally representative sample. Therefore, we down-sampled available sequences to \fpeval{\maxsamplingpercent}\% of confirmed case counts in each Swiss Canton each week between \mindate\ and \maxdate. Where there were not enough sequences available from a Canton in a week, we took all available sequences. To reduce the size of the alignments for phylogenetic analysis, we divided the focal Swiss set into Pango lineages \cite{Rambaut}, similar to \cite{DuPlessis2021}. Lineages composed of $>$ 50\% Swiss sequences were aggregated into their parent lineage(s) until $<=$ 50\% were Swiss. This aims to ensure that each analyzed lineage originated outside of Switzerland.

% describe nextstrain priority protocol for sim dataset
For the contextual sequences from abroad, we aimed to select the most genetically similar sequences to focal Swiss sequences. This set should help distinguish between SARS-CoV-2 variants unique to Switzerland (likely within-Switzerland transmission) and variants also circulating abroad (possibly recent introductions). To select this set, we applied the Nextstrain priority script \cite{Nextstrainteam} to rank sequences from abroad by their genetic similarity to Swiss sequences in each lineage alignment. Then, we selected \similaritycontextscalefactor\ times as many  context sequences as focal Swiss sequences for each analyzed lineage. 

% summarize the sample set
In the supporting information, we show sensitivity analyses for these choices. We tested different numbers of focal Swiss sequences and different ratios of contextual to focal sequences, with the conclusion that the most variation is still due to how we define an introduction (see ``Identifying introductions'', below). The final dataset used for analysis includes \nfocalsamples\ focal sequences from Switzerland and \nsimcontext\ genetically similar contextual sequences from abroad.

\subsection*{Timetree generation}
We estimated the maximum likelihood phylogeny for each lineage alignment using IQ-TREE \cite{Nguyen2014} under an HKY substitution model \cite{Hasegawa1985} with empirical base frequencies and 4 gamma rate categories. We then rooted each phylogeny with \outgroupgisaidepiisls\ as an outgroup and estimated branch lengths in time units using least-squares dating (LSD) \cite{To2016} with a strict molecular clock and a minimum mutation rate of 8x10-4 substitutions per site per year. We additionally assumed the root date to be between 15. Nov. and 24. Dec. 2019 (roughly in line with estimates provided by Nextstrain \cite{Nextstrainteam}) and set the minimum branch length to zero. Sequences that violated the strict clock assumption (z-score threshold > 3) were removed and near-zero branches ($<$1.7x10-5 substitutions per site per year) were collapsed into polytomies, reflecting the fact that the sequence data is not sufficient to resolve the ordering of these transmission events. Given the root date constraints, the mutation rate conformed to the lower bound of 8x10-4 with extremely narrow confidence intervals. Confidence intervals for node dates were generated in LSD by re- sampling branch lengths 100 times under a lognormal relaxed clock model with standard deviation 0.4.

\subsection*{Identifying introductions}
We ``picked'' putative Swiss transmission chains (collections of two or more genomic sequences resulting from within-Switzerland transmissions) off of each Pango lineage tree according to the following criteria: at least 2 Swiss sequences are part of a clade in the tree and the subtree spanned by these Swiss sequences is monophyletic upon removing (a) up to 3 export events where (b) only one export event may occur along each internal branch. Exports are defined to be clades containing non-Swiss sequences. We chose a conservative value for criterion (b) while still allowing some export events and note that the number of inferred transmission chains is robust to different values for criterion (a) given criterion (b) (Fig. \ref{fig:sensitivity_figs}c). For a sensitivity analysis on the importance of the specific values of criteria (a) and (b) see the supporting text.

We refer to any Swiss sequence not falling into a Swiss transmission chain as a Swiss singleton. We assume each singleton and each transmission chain represent an independent introduction of SARS-CoV-2 into Switzerland; together these are called introductions.

Finally, we repeated this picking procedure twice for each lineage tree, interpreting polytomies in two different ways. Once, we split Swiss clades descending from polytomies into independent introductions unless the polytomy only had a single non-Swiss descendent (see criterion (b)). Alternatively, we aggregated all Swiss clades descending from polytomies into a single introduction. These procedures represent two plausible extremes, yielding either many introductions or few introductions.

\subsection{Quantifying the effect of border closures on introductions}
We wanted to understand if the Swiss border closure on 13 Mar. 2020 changed introduction dynamics. Our null hypothesis is that the number of weekly introductions before border closure, estimated from the genome data as described above, is a linear function of weekly SARS-CoV-2 cases in surrounding countries. Introductions are dated by the first detected case, whereas case counts (taken from the ECDC \cite{ECDC}) are dated by case reporting dates. We considered an average delay between these variables up to 18 days. We also considered two options for ``surrounding countries'', either all the non-Swiss European countries in the ECDC dataset or just Switzerland's neighboring countries Italy, France, Germany, and Austria. We fit these models separately to the weekly many and few introductions estimates before 13 Mar. 2020 and selected the best-fit models (lowest root mean squared error). For both estimates, the neighboring-countries-only incidence data better predicted introductions. The best-fit delay was \casesdelaymax days for the many introductions estimates compared to \casesdelaymin days for the few introductions estimates. Figure \ref{fig:TODO} shows these best-fit models and projections if these dynamics had continued after 13. Mar.

\subsection{Quantifying the effect of lockdown conditions on persistence}
We also wanted to understand if the Swiss lockdown between 17 Mar. and 27 Apr. 2020 changed the persistence of introductions. Our null hypothesis is that the distribution of times until the currently circulating introductions die out does not depend on the day. In other words, introductions spreading on day $x$ will take just as long to die out as introductions spreading two weeks earlier, or at any other arbitrary point in time. We calculated the median and inter-quartile range of the time to the last sample across introductions persisting each date. Singleton introductions trivially persist for 1 day. Figure \ref{fig:TODO} shows how the persistence of the estimated introductions changes around the dates of the Swiss lockdown from 17. Mar to 27. Apr.

\subsection*{Phylodynamic analysis}

After identifying introductions, we performed phylodynamic inference on them using the BDSKY (birth-death skyline) method \cite{stadler2013birth} in BEAST2 \cite{Bouckaert2019}. To avoid model mis-specification due to the more transmissible alpha variant, we analyzed data only through 30 Nov.~2020.  The inference relies on two main models: a nucleotide substitution model describing an evolutionary process and a population dynamical model describing a transmission and sampling process. 

For the nucleotide substitution model, we assumed an HKY \cite{Hasegawa1985} model with 4 Gamma rate categories to account for site-to-site rate heterogeneity \cite{Yang1994}. We used the default priors for kappa and the scale factor of the Gamma distribution. We assumed a strict clock with rate $8\times 10^{-4}$ substitutions per site per year.

For the population dynamic model, we used BDSKY \cite{stadler2013birth}. In BDSKY, the identified introductions are the result of a birth-death with sampling process parameterized by an effective reproductive number, a becoming-uninfectious rate, and a sampling probability. As in \cite{Muller2020}, we inferred these population dynamical parameters jointly from the different introductions. More concretely, each introduction is assumed to result from an independent birth-death process having its own start time, but sharing all other parameters with the processes associated with the other introductions. 
Further, we used a uniform prior on the time of origin for each introduction, from 01. Jan until the end of the sampling period. This means our prior expectation is a uniform rate of introductions through time. We fixed the become-uninfectious rate to 36.5 per year, which corresponds to an average of 10 days to becoming uninfectious. We allowed the effective reproductive number to vary week-to-week. We applied an Ornstein-Uhlenbeck smoothing prior to the logarithm of this parameter, defined such that the stationary distribution of the process was $\text{LogNormal}(0.8, 0.5)$  and with an $\text{Exp}(1)$ hyperprior on the relaxation parameter of the process. This prior constrains both the absolute reproductive number and the relative sizes of the change between weeks. Finally, we allowed the sampling probability to vary when Swiss testing or genome sampling regimes changed significantly (Table S1). For our main analysis, we applied a $\text{LogUniform}(10^{-4}, 1)$ prior on the sampling probability. As a sensitivity analysis, we also tried a $\text{LogUniform}(10^{-4}, 0.05)$ prior since we upper-bounded our sampling to 5\% of confirmed cases each week. See the supporting text for more information.

Then, we added an additional ``contact tracing'' factor to the model. This factor is a multiplicative damping of the effective reproductive number applied to each introduction from 2 days after the first sample date until the introduction dies out. Since we hypothesized contact tracing was not functioning as well during periods of high case numbers, we estimated a separate value in each of three periods: before 15. Jun 2020 (Spring), 15 Jun.~to 30 Sept.~2020 (Summer), and 30. Sept to \maxdate\ (Fall). We used the same spike and slab prior for the damping factor in each period, with an inclusion probability of 0.5 and a uniform prior if included between 0 and 1. 

The XML files for the phylodynamic analysis are available on the project GitHub repository at TODO.

\acknow{This work was supported by the Swiss National Science Foundation (SNSF) through grant number 31CA30\_196267 (to TS). We gratefully acknowledge the authors from the originating laboratories responsible for obtaining the specimens and the submitting laboratories where genetic sequence data were generated and shared via the GISAID Initiative, on which this research is based. A full acknowledgements table of these groups, including the identifiers for all GISAID data used in this study, is available on the project GitHub repository at TODO. We also thank Jana Huisman for valuable discussions on the manuscript.}

\showacknow{} % Display the acknowledgments section

% Bibliography
\printbibliography

\end{document}
