% What added knowledge do genomes contribute beyond case data w/o metadata?
% Transmission chains - spatio-temporal patterns of spread (already done, we just summarize here)
% Transmission chains - degree of super-spreading (we don't currently address this)
% Transmission chains - efficacy of contact tracing (observational and modelling studies give a range of values for efficacy, depending on implementation success. We get at CH implementation success, possibly compared to other countries, here.)

\documentclass[9pt,twocolumn,twoside,lineno]{pnas-new}
% Use the lineno option to display guide line numbers if required.

\makeatletter
\newcommand*{\addFileDependency}[1]{% argument=file name and extension
  \typeout{(#1)}
  \@addtofilelist{#1}
  \IfFileExists{#1}{}{\typeout{No file #1.}}
}
\makeatother
\newcommand*{\myexternaldocument}[1]{%
    \externaldocument{#1}%
    \addFileDependency{#1.tex}%
    \addFileDependency{#1.aux}%
}
\usepackage{xr}
\myexternaldocument{manuscript/text_supporting}

\usepackage{todonotes}
% \usepackage{biblatex}
% \addbibresource{references.bib}

\input{manuscript/config_variables}
\input{manuscript/chain_summary_variables}
\input{manuscript/sample_summary_variables}

\templatetype{pnasresearcharticle} % Choose template 

\title{SARS-CoV-2 Genome surveillance in Switzerland: Can we quantify the effects of policy?}

% Use letters for affiliations, numbers to show equal authorship (if applicable) and to indicate the corresponding author
\author[a,c,1]{Author One}
\author[b,1,2]{Author Two} 
\author[a]{Author Three}

\affil[a]{Affiliation One}
\affil[b]{Affiliation Two}
\affil[c]{Affiliation Three}

% Please give the surname of the lead author for the running footer
\leadauthor{Lead author last name} 

% Please add a significance statement to explain the relevance of your work
\significancestatement{Authors must submit a 120-word maximum statement about the significance of their research paper written at a level understandable to an undergraduate educated scientist outside their field of speciality. The primary goal of the significance statement is to explain the relevance of the work in broad context to a broad readership. The significance statement appears in the paper itself and is required for all research papers.}

% Please include corresponding author, author contribution and author declaration information
\authorcontributions{Please provide details of author contributions here.}
\authordeclaration{Please declare any competing interests here.}
\equalauthors{\textsuperscript{1}A.O.(Author One) contributed equally to this work with A.T. (Author Two) (remove if not applicable).}
\correspondingauthor{\textsuperscript{2}To whom correspondence should be addressed. E-mail: author.two\@email.com}

% At least three keywords are required at submission. Please provide three to five keywords, separated by the pipe symbol.
\keywords{Keyword 1 $|$ Keyword 2 $|$ Keyword 3 $|$ ...} 

\begin{abstract}
% Please provide an abstract of no more than 250 words in a single paragraph. Abstracts should explain to the general reader the major contributions of the article. References in the abstract must be cited in full within the abstract itself and cited in the text.
Pathogen genome sequences can provide information on transmission chains and, potentially, how public health measures effect transmission. Here we use SARS-CoV-2 sequences from approximately \overallsamplingpercent\% of all confirmed cases in Switzerland from the first case on 25. Feb. through \maxdate\ to quantify transmission dynamics in Switzerland. We developed a two-step pyhylogenetic/phylodynamic analysis framework to analyze \nfocalsamples\ genomes from Switzerland in $\sim$12 hours. We report estimates across the range of plausible phylogenies, since sampling exceeds viral diversity and the SARS-CoV-2 phylogeny is uncertain. Using our approach, we show robust signal in SARS-CoV-2 transmission chains for a 40-60\% slow-down in transmission upon sampling when overall case counts are low. This effect is potentially due to test-trace-isolate measures, though we cannot exclude other unknown mechanisms. These analyses aim to help us understand the importance of international travel towards a national epidemic, an increasingly relevant topic as global vaccination campaigns continue at an uneven pace and new variants of concern spread internationally.
\end{abstract}

\dates{This manuscript was compiled on \today}
\doi{\url{www.pnas.org/cgi/doi/10.1073/pnas.XXXXXXXXXX}}

\begin{document}

\maketitle
\thispagestyle{firststyle}
\ifthenelse{\boolean{shortarticle}}{\ifthenelse{\boolean{singlecolumn}}{\abscontentformatted}{\abscontent}}{}

% \subsection*{Author Affiliations}

% Include department, institution, and complete address, with the ZIP/postal code, for each author. Use lower case letters to match authors with institutions, as shown in the example. PNAS strongly encourages authors to supply an \href{https://orcid.org/}{ORCID identifier} for each author. Individual authors must link their ORCID account to their PNAS account at \href{http://www.pnascentral.org/}{www.pnascentral.org}. For proper authentication, authors must provide their ORCID at submission and are not permitted to add ORCIDs on proofs.

\section{Introduction}
% If your first paragraph (i.e. with the \dropcap) contains a list environment (quote, quotation, theorem, definition, enumerate, itemize...), the line after the list may have some extra indentation. If this is the case, add \parshape=0 to the end of the list environment.
% \dropcap{T}his PNAS journal template is provided to help you write your work in the correct journal format. Instructions for use are provided below. 

% Claim importance
\dropcap{A} crucial task is to evaluate public health measures aimed at slowing transmission of the SARS-CoV-2 virus. Such measures aim to prevent new introductions into a region and contain ongoing transmission chains, for example by quarantining incoming travellers or enforcing general social distancing. So far, mathematical models have helped estimate the effects of significant measures like lockdowns and travel bans using data on case or death counts and travel volumes, e.g. \cite{Flaxman2020, Tian2020}. However, these data do not provide resolution at the scale of individual introductions and subsequent transmission chains.

Detailed contact tracing data could help reconstruct introductions and transmission chains, but these data are difficult to collect and sensitive. Alternately, pathogen genome sequences collected from infected hosts can help \cite{Kraemer}. Fast-mutating RNA viruses accumulate mutations on the same time-scale as host-to-host transmission. For example a typical SARS-CoV-2 strain will accumulate $\sim$2 mutations per month along its $\sim$30,000 nucleotide-long genome \cite{Nextstrainteam}. These mutations allow us to construct a viral phylogeny that is an approximation of the host-to-host transmission tree. 

SARS-CoV-2 genomes were collected at unprecedented scale throughout 2020 \cite{Alm2020} and have been extensively used to characterize transmission dynamics. Phylogenetic approaches estimate events like introductions and transmissions directly from reconstructed phylogenies, while phylodynamic approaches couple a mechanistic model of transmission or migration to the phylogeny, allowing inference of population-level dynamics. For example, phylogenetic studies by \cite{Mallon2020, duPlessis2021} showed that national lock-downs during the early Irish and English epidemics  strongly reduced lineage diversity and size. \cite{Miller2020, Geoghegan2020a, Muller2020a} used phylodynamic models to show that non-pharmaceutical interventions (NPIs) and/or reductions in population mobility reduced transmission rates in Israel, New Zealand, and Washington State, USA. Finally, \cite{Ragonnet-Cronin2021} compared data across many regions and showed a correlation between death counts and the delay between (phylogenetically-estimated) epidemic start and implementation of strong NPIs. However, the utility of genome data for evaluating specific measures in different regions is still largely unexplored.

% Here we can distinguish between phylogenetic and phylodynamic approaches to this task. Phylogenetic approaches aim to identify events like introductions and transmissions directly from SARS-CoV-2 phylogenies, e.g. \cite{Lu2020, Eden2020, OudeMunnink2020, Bluhm2020}. Going a step further, phylodynamic approaches link branching times in the phylogeny to a population demographic model \cite{Grenfell2004}. In summary, phylogenetic methods estimate epidemiological events specific to the analyzed samples, while phylodynamic methods aim to estimate population-wide dynamics. Importantly, both approaches to genomic epidemiology can help us evaluate public health measures. 

% Indicate a gap
Using SARS-CoV-2 genomes to quantify the effect of public health measures is difficult. One problem is sampling bias: sequencing efforts vary by region and through time. Often, available samples are unrepresentative of the whole population \cite{Villabona-Arenas2020, DeMaio2015}. A second problem is phylogenetic uncertainty: SARS-CoV-2 sequencing was high compared to viral diversity, in particular during the early epidemic in Spring 2020. If many sequences are similar or identical, the phylogeny contains unresolved multifurcations and is only a coarse approximation of the underlying host-to-host transmission tree \cite{Villabona-Arenas2020}. Arbitrarily resolved phylogenies may yield misleading or overly confident estimates \cite{Morel2021}. A third problem is model mis-specification: real-world transmission dynamics are governed by complex population structure. Highly parameterized phylodynamic models as in \cite{Miller2020, Geoghegan2020a, Muller2020a} can help - multiple compartments account for sampling biases and population structure, and the phylogeny is integrated out as a nuisance parameter. These models are generally too computationally complex to fit to thousands of genomes, though recent developments are speeding them up \cite{Lemey2021}. 
% In the absence of strong prior information, these models are also too richly parameterized to be identifiable \cite{Louca2021FundamentalEpidemiology}.

% State that our paper fills the gap 
Here we present a two-step analysis framework carefully combing phylogenetic and phylodynamic approaches to  address these difficulties and analyze thousands of SARS-CoV-2 sequences in $\sim$12 hours. In each step, we condition on monophyletic partitions of the sample set, an approach inspired by  \cite{Muller2020} and \cite{DuPlessis}. This ensures computational feasibility. To limit sampling biases, we utilize sequences collected as part of the nation-wide Swiss SARS-CoV-2 genomic surveillance program. Namely, we sub-sample  available sequences to be more spatio-temporally representative of the Swiss epidemic. Finally, we conservatively report estimates across the range of plausible phylogenies.

Using our framework, we aim to quantify transmission dynamics in Switzerland, with a particular focus on the effects of public health measures. We quantify a slow-down in transmission once transmission chains are sampled when case counts are low, which we hypothesize might be due to test-trace-isolate measures. While we focus on epidemiological insights from the Swiss epidemic, our framework can be applied to any regional SARS-CoV-2 epidemic with data available on GISAID \cite{GISAID}. We perform a small validation using New Zealand data to illustrate this. 

\section{Results}

\subsection{Introductions driving the Swiss epidemic}

We analyzed \nfocalsamples\ SARS-CoV-2 genomes collected in Switzerland, representing the Swiss epidemic from the first detected case on 25. Feb. through \maxdate. These data are a sub-sample of all available Swiss genomes aimed to be more spatio-temporally representative of the Swiss epidemic while maintaining a large sample size (Figure  \ref{fig:downsampling_representativeness}). 

First, we constructed an approximate maximum-likelihood phylogeny for each SARS-CoV-2 Pango lineage, including Swiss sequences and the most genetically similar foreign sequences available on GISAID \ref{tab:lineage-data-summary}. Within these lineage phylogenies, we identified introductions and subsequent transmission chains (hereafter: ``introductions'') as described in the Materials and Methods. Importantly, we generate estimates under two extreme resolutions of phylogenetic uncertainty, which allows us to report estimates across the range of strictly bifurcating phylogenies consistent with the genome data. We refer to these extremes as the ``smallest'' and ``largest'' plausible introductions.

We estimate the analyzed samples came from between \nchainsmin\ and \nchainsmax\ independent introductions into Switzerland. These introductions are roughly power law-distributed in size, with the 10 largest introductions accounting for \maxlargestchainsper\ to \minlargestchainsper \% of genome samples \ref{fig:chain_size_dist}. From a down-sampling analysis, we see that we do not reach saturation - if we were to include more genomes, we would identify more introductions \ref{fig:sensitivity_downsampling}. 

% For comparison: UK transmission lineages have power-law decay w/ alpha = 0.79 for chains 2 <= x <= 50, 1.26 for chains > 50. 
% Similar to Louis, I see chain sizes as power-law distributed, with a change in scale factor at ~ chain size 50. 
% The scaling factors I estimate are higher than his though (~1.2 for <= 50 and ~1.4 for >50). So, our Swiss chain sizes decay more sharply (fewer larger chains) than his UK transmission lineages.

The absolute number of introductions is very uncertain due to phylogenetic uncertainty and incomplete sampling. However, the trends through time in new introductions versus persistent transmission chains are more robust (Figure \ref{fig:chain-longevity}). A few transmission chains were quite persistent, including \nspanningchainsfebnovmin\ that may have persisted from the start of the Swiss epidemic in Feb. 2020 until Nov. 2020. But on average, Swiss introductions persisted \meantimetolastsamplemax\ - \meantimetolastsamplemin\ days from first to last sampling. Newly sampled introductions dropped from Mar. to Apr. 2020 and rose again from Jun. to Jul. 2020 (Figure \ref{fig:chain-longevity}). The Swiss borders were largely closed from 25. Mar. to 14. Jun. 2020. Finally, we can roughly estimate when introductions arrived as the date of the samples' most recent common ancestor (for transmission chains) or the sample date (for singletons). Introductions estimated to have arrived in the Swiss lockdown from 17. Mar. to 27. Apr. 2020 were more often singletons (63 - 85\% of introductions) than outside this period (31 - 72\%) \ref{tab:lockdown-contingency}.

\begin{figure}[tbhp]
\centering
\includegraphics[width=.8\linewidth]{figures/chain_longevity_matrix.png}
\caption{Number of sampled introductions by the month they were fist sampled, in Switzerland. Estimate ranges are between analyses assuming the largest and smallest plausible introductions. The red color scales with the log of the midpoint of the range. So, darker red means more sampled introductions and lighter red means fewer.}
\label{fig:chain-longevity}
\end{figure}

\subsection{Transmission dynamics within Switzerland}

Next, we jointly inferred time-varying transmission dynamics of the Swiss epidemic in a Bayesian phylodynamic framework. To test robustness, we repeated this analysis under several different model configurations (Table \ref{tab:phylo-models}). For each model, we replicated the analysis conditioning on either the smallest or largest plausible introductions. 

The basic birth-death skyline model we used is parameterized by a time-varying sampling probability and a time-varying effective reproductive number, which is a function of transmission and becoming-uninfectious rate parameters. We fixed the becoming-uninfectious rate, so the effective reproductive number varies only with the transmission rate. Here, we do not focus on estimating the effective reproductive number or the sampling proportion. Instead, we concentrate on insights that can only be generated by considering individual transmission chains (below). The effective reproductive number can be well-estimated using confirmed case data in Switzerland \cite{shinyRe}. Further, we argue the sampling proportion should be treated as a fitting parameter, since testing strategies varied through time in Switzerland and information from serological studies in different regions and at different time-points are difficult to harmonize. However, we did test for robustness to a strong prior on the sampling probability based on confirmed case counts (see Supporting Text).

We included an additional factor in the model to test whether test-trace-isolate measures in Switzerland succeeded in slowing transmission. We reason that test-trace-isolate programs cannot have an effect before an introduction is identified, but ideally should help slow transmission shortly after the first case tests positive. So, along with the baseline time-varying effective reproductive number, we introduced a reduction factor to quantify if and how much transmission decreased beginning 2 days post-sampling for each transmission chain.  We used a spike-and-slab prior on this factor to include the possibility of no transmission slow-down. We estimate a strong slow-down in transmission after chain detection during Summer 2020. This result is robust to phylogenetic uncertainty and whether or not the sampling proportion is upper-bounded based on confirmed case counts. In contrast, the data favor a model in which transmission is not reduced from the baseline post-sampling during Fall 2020. Estimates in Spring 2020 are not robust to phylogenetic uncertainty. This could be explained by low genomic diversity in SARS-CoV-2 at this time contributing to high phylogenetic uncertainty.

\subsection{Comparision to the New Zealand epidemic}
While Switzerland is centrally located in Europe and well-connected to other countries, especially those in the (normally) barrier-free Schengen travel zone, New Zealand is a relatively isolated island nation. Additionally, New Zealand aimed to eradicate SARS-CoV-2 throughout 2020 using strong measures, such as keeping it's borders closed, while Switzerland re-opened to Europe in early Summer 2020. We compared our estimates for the transmission slow-down factor between Switzerland in Spring, Summer, and Fall 2020 with New Zealand before and after local transmission was all but eradicated there in mid-May 2020. Similar to Switzerland, estimates for New Zealand are not robust to phylogenetic uncertainty prior to 15. May. After 15. May we estimate a similar or stronger transmission chain slow-down in New Zealand compared to Switzerland. 

\begin{figure}[tbhp]
\centering
\includegraphics[width=.8\linewidth]{figures/CT_conditionedDamping_noSampUB.pdf}
\caption{``Contact tracing'' scale factor estimated in three different time periods under the largest and smallest plausible introductions scenarios.}  
\label{fig:scale-factor}
\end{figure}

% \begin{itemize}
%     \item Spike and slab prior for Re multiplier
%     \item Bounded time-frame for Re multiplier
%     \item Sampling proportion upper bound 0.01 instead of 0.1 - Tim will rerun
%     \item Prior on cluster origins is not unique to each cluster, it's a big lognormal blob throughout 2020 - Tim will switch to unif(Jan 1 - Dec 31) so that our prior assumption is that the rate of introductions is constant from when the pandemic started spreading internationally until end of sampling period
%     \item How indep are Re estimates from phylo when I select seqs based on conf case \#s? - Tanja's not concerned, the genomes can still tell us what they tell us within what we already know from cases
%     \item 50\% prior y/n need Re factor, if y then unif(0, 1) on the factor 
%     \item Tim puts independent spike & slab priors in spring, summer, and fall and infers three Re reduction factors so we can see if Summer is really different
%     \item Remember Re plot is now background Re (without contact tracing) and no longer the actual average Re
% \end{itemize}

\section{Discussion}
\begin{itemize}
    \item Quantified effects of policy:
    \begin{itemize}
        \item Introductions were 47 - 51\% less likely to have spread to $>1$ sampled individual during the Swiss lockdown compared to outside the lockdown period.
        \item Reproductive number of introductions dropped 40 - 60\% after sampling during Summer 2020, possibly due to test-trace-isolate success upon case detection. 
    \end{itemize}
    \item Estimates on introductions and transmission chains are sensitive to phylogenetic uncertainty
    \begin{itemize}
        \item TODO: How does this compare to other studies?
        \item Recommend caution in trying to show who-infected-who when transmission timescale faster than mutational timescale
    \end{itemize}
    \item We treat the time-varying sampling proportion as a fitting parameter
    \begin{itemize}
        \item The temporal trends make sense according to know changes in testing and genome sampling programs
        \item Exception is initially high sampling proportion and unrealistically high end sampling proportion
        \item Cite supplemental figure where we upper-bounded the sampling proportion and discuss that results consistent except for double-spike at end
    \end{itemize}
    \item Contact tracing at the national level isn't as effective for new introductions.
    \begin{itemize}
        \item We find robust signal for decrease in transmission after chain detection in Summer 2020 compared to Spring 2020. Fall 2020 inference is not robust to phylogenetic uncertainty. 
        \item This slow correlates with the approximate start to contact tracing efforts, but an alternate explanation could be that sampled transmission chains originate in early super-spreading events.
    \end{itemize}
    \item Novel insights on patterns of SARS-CoV-2 spread?
    \item Are our transmission chains geographically constrained? How do we explain if not?
    \item Address transmission dynamics with regard to lockdown and travel restriction policies
\end{itemize}
% MAJOR QUESTION: was the summer epidemic growing in size due to travel (ie import of strains which seeded local transmission chains)?
% Contact tracing factor could also be catching superspreading events that found transmission chains

\section{Acknowledgments}



% \subsection*{Submitting Manuscripts}

% All authors must submit their articles at \href{http://www.pnascentral.org/cgi-bin/main.plex}{PNAScentral}. If you are using Overleaf to write your article, you can use the ``Submit to PNAS'' option in the top bar of the editor window. 

% \subsection*{Format}

% Many authors find it useful to organize their manuscripts with the following order of sections;  title, author line and affiliations, keywords, abstract, significance statement, introduction, results, discussion, materials and methods, acknowledgments, and references. Other orders and headings are permitted.

% \subsection*{Manuscript Length}

% A standard 6-page article is approximately 4,000 words, 50 references, and 4 medium-size graphical elements (i.e., figures and tables). The preferred length of articles remains at 6 pages, but PNAS will allow articles up to a maximum of 12 pages.

\subsection*{References}
\printbibliography

% References should be cited in numerical order as they appear in text; this will be done automatically via bibtex, e.g. \cite{belkin2002using} and \cite{berard1994embedding,coifman2005geometric}. All references cited in the main text should be included in the main manuscript file.

% \subsection*{Data Archival}

% PNAS must be able to archive the data essential to a published article. Where such archiving is not possible, deposition of data in public databases, such as GenBank, ArrayExpress, Protein Data Bank, Unidata, and others outlined in the \href{https://www.pnas.org/page/authors/journal-policies#xi}{Information for Authors}, is acceptable.

% \subsection*{Language-Editing Services}
% Prior to submission, authors who believe their manuscripts would benefit from professional editing are encouraged to use a language-editing service (see list at www.pnas.org/page/authors/language-editing). PNAS does not take responsibility for or endorse these services, and their use has no bearing on acceptance of a manuscript for publication. 

% \begin{figure}%[tbhp]
% \centering
% \includegraphics[width=.8\linewidth]{frog}
% \caption{Placeholder image of a frog with a long example legend to show justification setting.}
% \label{fig:frog}
% \end{figure}


% \begin{SCfigure*}[\sidecaptionrelwidth][t]
% \centering
% \includegraphics[width=11.4cm,height=11.4cm]{frog}
% \caption{This legend would be placed at the side of the figure, rather than below it.}\label{fig:side}
% \end{SCfigure*}

% \subsection*{Digital Figures}

% EPS, high-resolution PDF, and PowerPoint are preferred formats for figures that will be used in the main manuscript. Authors may submit PRC or U3D files for 3D images; these must be accompanied by 2D representations in TIFF, EPS, or high-resolution PDF format. Color images must be in RGB (red, green, blue) mode. Include the font files for any text.

% Images must be provided at final size, preferably 1 column width (8.7cm). Figures wider than 1 column should be sized to 11.4cm or 17.8cm wide. Numbers, letters, and symbols should be no smaller than 6 points (2mm) and no larger than 12 points (6mm) after reduction and must be consistent. 

% Figures and tables should be labelled and referenced in the standard way using the \verb|\label{}| and \verb|\ref{}| commands.

% Figure \ref{fig:frog} shows an example of how to insert a column-wide figure. To insert a figure wider than one column, please use the \verb|\begin{figure*}...\end{figure*}| environment. Figures wider than one column should be sized to 11.4 cm or 17.8 cm wide. Use \verb|\begin{SCfigure*}...\end{SCfigure*}| for a wide figure with side legends.

% \subsection*{Tables}
% Tables should be included in the main manuscript file and should not be uploaded separately.

% \subsection*{Single column equations}

% Authors may use 1- or 2-column equations in their article, according to their preference.

% To allow an equation to span both columns, use the \verb|\begin{figure*}...\end{figure*}| environment mentioned above for figures.

% Note that the use of the \verb|widetext| environment for equations is not recommended, and should not be used. 

% \begin{figure*}[bt!]
% \begin{align*}
% (x+y)^3&=(x+y)(x+y)^2\\
%       &=(x+y)(x^2+2xy+y^2) \numberthis \label{eqn:example} \\
%       &=x^3+3x^2y+3xy^3+x^3. 
% \end{align*}
% \end{figure*}


% \begin{table}%[tbhp]
% \centering
% \caption{Comparison of the fitted potential energy surfaces and ab initio benchmark electronic energy calculations}
% \begin{tabular}{lrrr}
% Species & CBS & CV & G3 \\
% \midrule
% 1. Acetaldehyde & 0.0 & 0.0 & 0.0 \\
% 2. Vinyl alcohol & 9.1 & 9.6 & 13.5 \\
% 3. Hydroxyethylidene & 50.8 & 51.2 & 54.0\\
% \bottomrule
% \end{tabular}

% \addtabletext{nomenclature for the TSs refers to the numbered species in the table.}
% \end{table}

% \subsection*{Supporting Information Appendix (SI)}

% Authors should submit SI as a single separate SI Appendix PDF file, combining all text, figures, tables, movie legends, and SI references. SI will be published as provided by the authors; it will not be edited or composed. Additional details can be found in the \href{https://www.pnas.org/authors/submitting-your-manuscript#manuscript-formatting-guidelines}{PNAS Author Center}. The PNAS Overleaf SI template can be found \href{https://www.overleaf.com/latex/templates/pnas-template-for-supplementary-information/wqfsfqwyjtsd}{here}. Refer to the SI Appendix in the manuscript at an appropriate point in the text. Number supporting figures and tables starting with S1, S2, etc.

% Authors who place detailed materials and methods in an SI Appendix must provide sufficient detail in the main text methods to enable a reader to follow the logic of the procedures and results and also must reference the SI methods. If a paper is fundamentally a study of a new method or technique, then the methods must be described completely in the main text.

% \subsubsection*{SI Datasets} 

% Supply .xlsx, .csv, .txt, .rtf, or .pdf files. This file type will be published in raw format and will not be edited or composed.


% \subsubsection*{SI Movies}

% Supply Audio Video Interleave (avi), Quicktime (mov), Windows Media (wmv), animated GIF (gif), or MPEG files. Movie legends should be included in the SI Appendix file. All movies should be submitted at the desired reproduction size and length. Movies should be no more than 10MB in size.


% \subsubsection*{3D Figures}

% Supply a composable U3D or PRC file so that it may be edited and composed. Authors may submit a PDF file but please note it will be published in raw format and will not be edited or composed.


\matmethods{

\subsection*{Genome sequencing and quality filtering}
Most of the Swiss SARS-CoV-2 genomes analyzed until 31. Dec 2020 were generated by the Swiss SARS-CoV-2 Sequencing Consortium. Here, we briefly describe the swab-to-sequence process for these samples. RNA extracts from qPRC-positive patient naso- or oral-pharangeal swabs were provided by Viollier AG, a Swiss medical diagnostics company. RNA extraction was done with either the Abbott m2000sp or Seegene STARMag 96x4 Universal Cartridge RNA extraction kit. Extracts were then  transferred to the Genomics Facility Basel or the Functional Genomics Center Zurich for whole-genome sequencing. Both centers used the ARCTIC v3 primer scheme \cite{Quick2017, ARCTICNetwork} to generate tiled, approximately 400bp-long amplicons. Library preparation was done with the New England Biolabs (NEB) library preparation kit. Libraries were sequenced on Illumina MiSeq or NovaSeq machines, resulting in 2 x 251 base reads. Bioinformatics was done using V-pipe \cite{Posada-Cespedes2020}, including read trimming and filtering with PRINSEQ \cite{Schmieder2011}, alignment to Genbank accession MN908947 \cite{Wu2020} with bwa \cite{Li2009}, and consensus base calling. For consensus base calling, positions with $<$5x coverage are masked with ``N'', positions with $>$5\% and $>$2 reads supporting a minor base are called with IUPAC ambiguity codes, and positions with $>$50\% reads supporting a deletion are called with ``-''. We rejected samples with $<$20,000 non-N bases. The consensus sequences we generated have been made publicly available on both GISAID (submitting lab: Department of Biosystems Science and Engineering, ETH Zürich) and ENA (study: PRJEB38472).

We supplemented our Swiss data with other Swiss sequences and foreign sequences available via GISAID (accessed 31. May 2021). From the full set of sequences available on GISAID, we removed from consideration non-human samples, samples $<$ 27000 bases long, and samples flagged by the Nextclade tool \cite{Aksamentov} for one of the following reasons: suspiciously clustered SNPs (QC SNP clusters status metric not ``good''; $>=$ 6 mutations in 100 bases), too many private mutations (QC private mutations status metric not ``good''; $>=$ 10 mutations from the nearest tree node), or overall bad quality (Nextclade QC overall status ``bad''). We aligned the sequences to the reference genome MN908947.3 using MAFFT \cite{katoh_mafft:_2002}. Finally, we followed the Nextstrain pipeline's recommendation to mask the first 100 and last 50 sites of the alignment \cite{Nextstraina} since the start and end of SARS-CoV-2 sequences are prone to sequencing error \cite{DeMaio2020}.

\subsection*{Sampling procedure}

From the quality-filtered alignment of GISAID sequences, we selected a focal set of sequences from Switzerland and a context set of sequences from abroad. 

% describe representativeness of Swiss dataset here too?
For the focal sequences from Switzerland, we aimed to select a spatially and temporally representative sample. Therefore, we down-sampled available sequences to \maxweeklysamplingpercent\% of confirmed case counts in each Swiss Canton each week between \mindate\ and \maxdate. Where there were not enough sequences available from a Canton in a week, we took the maximum available sequences. To reduce the size of the alignments for phylogenetic analysis, we divided the focal Swiss set into Pango lineages \cite{Rambaut}, similar to \cite{DuPlessis2021}. Lineages composed of $>=$ 50\% Swiss sequences were aggregated into their parent lineage(s) until $<$ 50\% were Swiss. This aims to ensure that each analyzed lineage originated outside of Switzerland.

% describe nextstrain priority protocol for sim dataset
For the context sequences from abroad, we aimed to select the most genetically similar sequences to focal Swiss sequences. This set should help distinguish between SARS-CoV-2 strains unique to Switzerland (likely within-Switzerland transmission) and strains also circulating abroad (possibly recent introductions). To select this set, we applied the Nextstrain priority script \cite{Nextstrain} to rank sequences from abroad by their genetic similarity to Swiss sequences in each lineage alignment. Then, we selected \gensimscalefactor times as many most context sequences as focal Swiss sequences for each analyzed lineage. 

% describe context set estimation
% To generate the context dataset, we selected \travelcontextscalefactor\ times as many foreign genomes as Swiss genomes proportionally to monthly incidence- and travel-based estimates of SARS-CoV-2 introductions and irregardless of Pango lineage. Estimated introductions were based on case exposure locations recorded by the Swiss Federal Office for Public Health (FOPH), data on tourist arrivals at hotels published by the Swiss Federal Statistical Office (FSO), and data on the number of cross-border commuter permit holders published by the FSO. Here we aim to construct a reasonable prior estimate for monthly introductions into Switzerland and recognize that these estimates are sensitive to our subjective weighting of various data sources. Therefore, we interpret our geographic inferences in the context of these assumptions.

% Given our estimates for monthly SARS-CoV-2 introductions, we randomly selected genomes available on GISAID as of \GISAIDpulldate\ proportional to these estimates. For months without enough genomes from a country, we took extra sequences from the following month if possible and the preceding month if necessary. 

% describe sensitivity analyses here
% We applied this sub-sampling protocol three times with different random seeds. As an additional sensitivity check, we also padded the estimated introductions per country and month by 1 before determining the number of context sequences to take. Based on the padded numbers, we randomly chose another set of context sequences. In all, we analyzed three replicate context datasets and one context dataset based on padded introductions. In the main text, we discuss results based on only one of the replicate datasets; results from the other datasets are qualitatively similar (Fig. S8 – S10).

% summarize the sample set
The final dataset for analysis includes \nfocalsamples\ focal sequences from Switzerland and \nsimcontext\ genetically similar context sequences from abroad.

\subsection*{Timetree generation}
We estimated the maximum likelihood phylogeny for each lineage alignment using IQ-TREE \cite{Nguyen2014} under an HKY substitution model \cite{Hasegawa1985} with empirical base frequencies and 4 gamma rate categories. We then rooted each phylogeny with \outgroupgisaidepiisls\ as an outgroup and estimated branch lengths in time units using least-squares dating (LSD) \cite{To2016} with a strict molecular clock and a minimum mutation rate of 8x10-4 substitutions per site per year. We additionally assumed the root date to be between 15. Nov. and 24. Dec. 2019 (roughly in line with estimates provided by Nextstrain \cite{Nextstrainteam}) and set the minimum branch length to zero. Sequences that violated the strict clock assumption (z-score threshold > 3) were removed and near-zero branches ($<$1.7x10-5 substitutions per site per year) were collapsed into polytomies, reflecting the fact that the sequence data is not sufficient to resolve the ordering of these transmission events. Given the root date constraints, the mutation rate conformed to the lower bound of 8x10-4 with extremely narrow confidence intervals. Confidence intervals for node dates were generated in LSD by re- sampling branch lengths 100 times under a lognormal relaxed clock model with standard deviation 0.4.

\subsection*{Phylogenetic analysis}
We ``picked'' putative Swiss transmission chains off of each Pango lineage tree according to the following criteria: at least 2 Swiss sequences are part of a clade in the tree and the subtree spanned by these Swiss sequences is monophyletic upon removing (a) up to 3 export events where (b) only one export event may occur along each internal branch. Exports are defined to be clades containing non-Swiss sequences. We chose a conservative value for criterion (b) while still allowing some export events and note that the number of inferred transmission chains is robust to different values for criterion (a) given criterion (b) (Fig. S13).

We repeated our analysis interpreting polytomies in two ways. Once, we split Swiss clades descending from polytomies unless the polytomy only had a single non-Swiss descendent (see criterion (b)). Alternatively, we aggregated all Swiss clades descending from polytomies into a single transmission chain. These procedures represent two plausible extremes, the first being maximum introductions and minimum local transmission and the second being minimum introductions and maximum local transmission.

We refer to any Swiss sequence not falling into a Swiss transmission chain as a Swiss singleton. We assume each singleton and each transmission chain represent an independent introduction of SARS-CoV-2 into Switzerland; together these are called introductions.

% Date of each introduction?

% \subsection*{Transmission chain origin estimation}
% We inferred the source location of each introduction with a parsimony-based approach. For this, we used only tips in the context dataset and ignored tips in the similarity dataset. Otherwise, over-sequenced locations would show up too often as sources (24). Our procedure is as follows: we begin by labelling the MRCAs of Swiss transmission chains and all subtending nodes in the tree (excepting exported clades) as Swiss. Switzerland was excluded as a possible location for all other nodes. Given these constraints, we calculate the parsimony score for each possible location at each remaining node. Among the possible locations, we also include a “dummy” location to record the maximum parsimony score. So, in a first step we calculate parsimony scores up the tree. In a second step, we covert the parsimony scores at each node to location weights by taking the difference to the “dummy” score. Thus, locations with no support in the subtending tree are given a weight 0 and supported locations are weighted by the number of location changes they prevent. Importantly, these weights are determined through the parsimony score of the subtree descending the considered node and not of the full tree. Finally, we normalized the scores to 1 for each node. In summary, locations that would require more state changes in the subtending tree are down-weighted. For the summary figure Fig. 2b, we conservatively count each polytomy with Swiss descendants as a single introduction.

% \subsubsection{Test for travel quarantine effect}

\subsection*{Phylodynamic analysis}

We jointly inferred the effective reproductive number and the sampling proportion from all introductions as in \cite{Muller2020}. We performed inference using the BDSKY model \cite{stadler2013birth} in BEAST2 \cite{Bouckaert2019}, which assumes a birth-death population dynamical model. Each introduction is assumed to result from an independent birth-death process having its own start time, but sharing all other parameters with the processes associated with the other introductions. We fixed the expected become-uninfectious rate to 36.5 per year, which corresponds to an average time to becoming uninfectious of 10 days. We assumed an HKY \cite{Hasegawa1985} nucleotide substitution model with 4 gamma rate categories to account for site-to-site rate heterogeneity \cite{Yang1994}. We assumed a strict clock with a rate fixed to 8x10-4 substitutions per site per year. We applied a uniform prior from 01. Jan to 31. Dec 2020 to the time of origin for each introduction, so that we are assuming a uniform rate of introductions through time. The effective reproductive number (Fig. \ref{}) was allowed to vary according to an Ornstein-Uhlenbeck prior with a LogNormal(0.8, 0.5) stationary distribution. An Exp(1) hyperprior was applied to the variance of the Gaussian component of the smoothing prior. This prior constrains both the relative sizes of the change between weeks and the absolute reproductive number. The sampling proportion (Fig. ) was allowed to vary at time points when Swiss testing or genome sampling regimes changed significantly (Table \ref{}). An ??? prior was applied to the sampling proportion.

We set up an additional analysis with one additional parameter to test our hypothesis that contact tracing reduced the effective reproductive number shortly after each introduction was discovered. The ``contact tracing factor'' parameter is a multiplicative scaling of the effective reproductive number applied to each introduction from 2 days after the first sample date until the last sample in the introduction. Since we suspected contact tracing was not functioning as well during periods of high case loads, we estimated a separate value in each of three periods: before 15. Jun 2020, 15. Jun to 30. Sept 2020, and 30. Sept to 31. Dec 2020. The same spike and slab prior was applied to the contact tracing factor in each period, with a prior inclusion probability of ?? and a uniform prior if included between 0 and 1. The XML file including these modification is available at TODO.

}

\showmatmethods{} % Display the Materials and Methods section

\acknow{This work was supported by the Swiss National Science Foundation (SNSF) through grant number 31CA30\_196267 (to TS).
}

\showacknow{} % Display the acknowledgments section

% Bibliography
\bibliography{references}

\end{document}
