\documentclass[9pt,twoside,lineno]{pnas-new}
% Use the lineno option to display guide line numbers if required.

\templatetype{pnassupportinginfo}
\usepackage{longtable}
% \usepackage{biblatex}
% \addbibresource{references.bib}

\title{Your main manuscript title}
\author{Author1, Author2 and Author3 (complete author list)}
\correspondingauthor{Corresponding Author name.\\E-mail: author.two@email.com}

% \input{manuscript/chain_summary_variables}

\begin{document}

%% Comment out or remove this line before generating final copy for submission; this will also remove the warning re: "Consecutive odd pages found".
% \instructionspage  

\maketitle 

%% Adds the main heading for the SI text. Comment out this line if you do not have any supporting information text.
\SItext


% \subsection*{Subhead}
% Type or paste text here. This should be additional explanatory text such as an extended technical description of results, full details of mathematical models, etc.   
\section{Sensitivity analyses}

In this section, we describe our decision-making process to choose parameters for our phylogenetic analysis and report on the sensitivity of our results to these choices. 

\subsection{Down-sampling Swiss sequences}
We aim to include Swiss sequences that are spatio-temporally representative of the Swiss epidemic. Therefore, we down-sample available Swiss sequences proportional to confirmed case counts at the regional (cantonal) level. To assess whether the number of transmission chains saturates as we add more and more sequences, we did a saturation analysis. TODO: add figure and discuss implications.

\subsection{Ratio of foreign to Swiss sequences}
We investigated the sensitivity of transmission chain summary statistics to different ratios of genetically similar context sequences from abroad to focal Swiss sequences. Figure \ref{fig:sensitivity_context_set_size} shows that increasing the ratio of context to focal sequences from 1:1 to 3:1 increases the number of estimated transmission chains and decreases their mean size. However, the greatest differences in estimated transmission chains result from the different assumptions about how to resolve phylogenetic uncertainty (smallest vs. largest plausible chains), not the ratio of foreign to Swiss sequences. Therefore, we chose the 2:1 ratio as a balance of speed (smaller dataset = faster tree search convergence) and information content (larger dataset = more precise estimates of transmission chain import dates). We rely on the difference between the smallest and largest plausible chains picked from one such tree to represent uncertainty in the transmission chains.

\subsection{Transmission chain criteria}
After deciding on a ratio of foreign to Swiss sequences, we investigated the sensitivity of transmission chain summary statistics to our criteria for picking transmission chains. Figure \ref{fig:sensitivity_m_p} shows that increasing the maximum number of exported lineages decreases the number of transmission chains and increases their size. Increasing the maximum consecutive exports has a much smaller effect. But again, these differences are smaller than those resulting from the different assumptions about how to resolve phylogenetic uncertainty (smallest vs. largest plausible chains). Therefore, we chose our criteria to be maximum 3 exported lineages and maximum one consecutive export to allow for some exports but not arbitrarily many. 

\subsection{Upper-bounding sampling proportion}
Genome-based estimates for the effective reproductive number in Switzerland are for the most part robust to the different transmission chain definitions and largely match estimates based on confirmed case counts (Figure \ref{fig:re}). One exception is in October 2020, when genome-based estimates are lower than confirmed case counts unless the sampling proportion is treated as a fitting parameter. A drop in genome diversity during this period might explain the discrepancy. 

\subsection{Including a ``super-spreader'' deme}

% \section*{Heading}
% \subsection*{Subhead}
% Type or paste text here. You may break this section up into subheads as needed (e.g., one section on ``Materials'' and one on ``Methods'').

% \subsection*{Materials}
% Add a materials subsection if you need to.

% \subsection*{Methods}
% Add a methods subsection if you need to.


% %%% Each figure should be on its own page
\begin{figure}
\centering
\includegraphics[width = 11.4cm]{figures/CHE_downsampling.png}
\caption{Spatio-temporal representativeness of analyzed genome sequences. The mirror y-axis aims to contrast temporal trends in confirmed cases (red) versus analyzed sequences (blue); all values are positive counts. Facet titles are standard abbreviations for Swiss cantons. 'CHE' represents Swiss-wide cases and sequences, since case count reporting by canton began mid-May 2020.}  
\label{fig:downsampling_representativeness}
\end{figure}

\begin{figure}
\centering
\includegraphics[width = 11.4cm]{figures/chain_size_dist.png}
\caption{Size distribution of estimated transmission chains.}  
\label{fig:chain_size_dist}
\end{figure}

\begin{figure}
\centering
\includegraphics[width = 11.4cm]{figures/fig_SX_sensitivity_subsampling.png}
\caption{Number of estimated introductions as a function of the fraction of Swiss confirmed cases analyzed.}  
\label{fig:sensitivity_downsampling}
\end{figure}

\begin{figure}
\centering
\includegraphics[width = 11.4cm]{figures/Samp_noSampUB.png}
\caption{Estimated sampling proportion from Bayesian phylodynamic analysis.}  
\label{fig:samp-prop}
\end{figure}

\begin{figure}
\centering
\includegraphics[width = 11.4cm]{figures/fig_SX_sensitivity_context_set_size.png}
\caption{Sensitivity of transmission chain summary statistics to different ratios of focal Swiss sequences to genetically similar foreign context sequences (1:1, 1:2, and 1:3).}  
\label{fig:sensitivity_context_set_size}
\end{figure}

\begin{figure}
\centering
\includegraphics[width = 11.4cm]{figures/fig_SX_sensitivity_chain_defn.png}
\caption{Sensitivity of transmission chain summary statistics to different definitions of a transmission chain.}  
\label{fig:sensitivity_m_p}
\end{figure}

\newpage
\begin{table}
\caption{Phylodynamic models.}
\label{tab:phylo-models}
\begin{tabular}{cccccc}
Data set & Polytomy resolution & Base model & Damping factor & Damping factor + 2-Deme & Damping factor + upper-bounded $s$ \\
\midrule
Swiss & Maximum introductions & Y & Y & Y & Y \\
Swiss & Minimum introductions & Y & Y & Y & Y \\
New Zealand & Maximum introductions & Y & Y & & \\
New Zealand & Minimum introductions & Y & Y & & \\
\bottomrule
\end{tabular}
\end{table}

\begin{figure}[tbhp]
\centering
\includegraphics[width=.8\linewidth]{figures/Re_noSampUB.pdf}
\caption{Effective reproductive number estimated under the largest and smallest plausible introductions scenarios in analyses with and without a ``contact tracing'' scale factor.}  
\label{fig:re}
\end{figure}


\newpage
\begin{table}
\caption{Contingency table for singleton introductions and transmission chains by time period for smallest (right) and largest (left) plausible introductions.}
\input{figures/max_chains_contingency}
\input{figures/min_chains_contingency}
\label{tab:lockdown-contingency}
\end{table}

\newpage
\begin{table}%[tbhp]
\centering
\caption{Sampling proportion change-points in the Bayesian phylodynamic analysis.}
\begin{tabular}{lp{6cm}rr}
Start date & Description & Reference & TODO: Estimated sampling proportion \\
\midrule
 & Epidemic start & $-$ & 0.1 \\
23. Apr 2020 & All symptomatic individuals can get tested &  & 0.03 \\
25. Jun 2020 & Government pays for tests for symptomatic individuals &  & 0.08 \\
14. Sept 2020 & Genome sampling $<<$ 5\% of confirmed cases & & 0.001 \\
28. Sept 2020 & Number of tests conducted and \% positivity dramatically increase, genome sampling also increases & & 0.1 \\
19. Oct 2020 & Genome sampling $<<$ 5\% of confirmed cases again & & 0.06 \\
11. Nov 2020 & Genome sampling improves again & & 0.45 \\
\bottomrule
\end{tabular}
% \newline
% \addtabletext{}
\end{table}

\newpage
\input{figures/lineage_data_summary}

% %%% Add this line AFTER all your figures and tables
% \FloatBarrier

% \movie{Type legend for the movie here.}

% \movie{Type legend for the other movie here. Adding longer text to show what happens, to decide on alignment and/or indentations.}

% \movie{A third movie, just for kicks.}

% \dataset{dataset_one.txt}{Type or paste legend here.}

% \dataset{dataset_two.txt}{Type or paste legend here. Adding longer text to show what happens, to decide on alignment and/or indentations for multi-line or paragraph captions.}

\bibliography{references}

\end{document}
