\documentclass[9pt,twoside,lineno]{pnas-new}
% Use the lineno option to display guide line numbers if required.

\templatetype{pnasresearcharticle}
\usepackage{longtable}
\usepackage{array,ragged2e} % for nicer justification of table entries with "P"
\newcolumntype{P}[1]{>{\RaggedRight\arraybackslash}p{#1}}
\usepackage{caption} % for subfigures
\usepackage{subcaption}
\usepackage{lscape} % for turning one page to landscape

\let\thefigureWithoutS\thefigure %% <- store old definition
\renewcommand\thefigure{S\thefigureWithoutS}

\usepackage[backend=bibtex,style=nature]{biblatex}
\bibliography{references}

\title{Supporting information}
\author{}
% \correspondingauthor{Corresponding Author name.\\E-mail: author.two@email.com}

\begin{document}

\maketitle 

\tableofcontents

\newpage

\section{Supporting information text}

% \subsection*{Subhead}
% Type or paste text here. This should be additional explanatory text such as an extended technical description of results, full details of mathematical models, etc.   
\subsection{Sensitivity analyses for identifying introductions}
Here we describe different sensitivity analyses we performed for the definition of an introduction. 

\subsubsection{Number of focal Swiss sequences analyzed}
To assess whether the number of identified introductions saturates as we add more sequences, we did a sub-sampling analysis. We sub-sampled the Swiss genome sequences used in our main analysis from 5\% of confirmed cases down to 1\% of confirmed cases. For each new sub-sampling value, we calculated the number of introductions (singletons and transmission chains) that we would have identified given the sub-sampled sequence set. Figure \ref{fig:sensitivity_figs}a shows that as we add sequences, we do not reach saturation. Therefore, if we were to include even more sequences, we would almost certainly identify more introductions into Switzerland.

\subsubsection{Ratio of foreign context to focal Swiss sequences analyzed}
Next, we assessed how summary statistics about identified introductions and the runtime of our pipeline changed as we added more and more foreign context sequences. We kept the number of focal Swiss sequences constant at 5\% of confirmed cases each week and added foreign context sequences at a 1:1, 2:1, and 3:1 ratio to focal Swiss sequences. Figure \ref{fig:sensitivity_figs}b shows that as we add foreign context sequences, we identify more numerous, smaller introductions. However, the greatest differences come from the different assumptions about how to resolve polytomies (few vs. many introductions), not the ratio of foreign context to focal Swiss sequences. Therefore, we chose to present results using the 2:1 ratio to balance speed (smaller dataset = faster tree search convergence) and information content (larger dataset = more precise estimates of introduction dates).

\subsubsection{Criteria for identifying introductions}
After deciding on a ratio of foreign context to focal Swiss sequences, we assessed how summary statistics about identified introductions changed depending on the precise heuristic definition of an introduction. Given phylogenetic trees with polytomies, we picked transmission chains off of these trees as described in the main text methods. Here, we varied (a) the number of export events allowed in each transmission chain and (b) the maximum number of export events allowed to occur along each single internal branch. Figure \ref{fig:sensitivity_figs}c shows that increasing (a) yields fewer, larger transmission chains. Increasing (b) for each level of (a) has a much smaller effect. We note that again, the greatest differences come from the different assumptions about how to resolve polytomies. Therefore, we chose to present results using a transmission chain definition based on (a) maximum 3 exported lineages and (b) maximum one consecutive export on each internal branch. This allows for some exports from Swiss transmission chains but not arbitrarily many. 

\subsection{Sensitivity analyses for phylodynamic modelling}
Here we describe a sensitivity analyses and some example intermediate outputs from our phylodynamic analysis.

\subsubsection{Sampling proportion prior}
We wanted to check that our phylodynamic estimates of a transmission damping factor are robust to our prior on the sampling proportion. Therefore, we repeated our analyses using two different priors on the sampling proportion. The first prior was a uniform distribution between 0 and 1. This broad prior allows the sampling proportion to assume any plausible value. The second prior was a uniform distribution between 0 and an upper bound equal to the number of genome sequences analyzed in a week divided by the number of confirmed cases that week. This narrower prior assumes that there are at least as many cases each week as confirmed cases. Figure \ref{fig:ReSampProbResults}a shows that in Switzerland, the estimated sampling proportion in late Fall 2020 varies greatly depending on the prior. A drop in SARS-CoV-2 diversity in Switzerland during this period might explain why the inference under the broader sampling prior estimates a proportion corresponding to fewer individuals than we know were infected during this time. Figure \ref{fig:ReSampProbResults}b shows that the effective reproductive number estimates in Fall 2020 for Switzerland more closely match estimates based on line-list data when the sampling proportion is treated as a fitting parameter, i.e. under the first, broad prior. Therefore, we report results under this prior in the main text. In Figure \ref{fig:DampingFactorResults}a, we show that the damping factor results are qualitatively similar between the two sampling proportion priors. 

\subsubsection{Logged trees}
Finally, we logged phylogenetic trees for a few introductions. These trees were sampled by the Markov chains in the phylodynamic analyses. Note that the damping factor results are jointly inferred from all the branching events across introductions in each time period. But since logging all introductions would yield huge log files, we logged just a few individual introductions under each set of model assumptions for visual inspection. For each set of model assumptions and each month, we logged trees for the 50th and 95th percentile largest introductions that were first sampled that month and eventually yielded > 2 samples. Figure \ref{fig:logged-chains} shows as an example summary trees for these introductions under the phylodynamic analysis for Switzerland with an un-bounded sampling proportion prior and a homogeneous population (no class of individuals with more contacts).

\newpage

\section{Supporting information figures}

\begin{figure}[h!]
\centering
\includegraphics[width = 0.8\linewidth]{figures/CHE_downsampling.png}
\caption{Spatio-temporal representativeness of analyzed genome sequences. The mirror y-axis aims to contrast temporal trends in confirmed cases (red) versus analyzed sequences (blue); all values are positive counts. Facet titles are standard abbreviations for Swiss cantons. 'CHE' represents Swiss-wide cases and sequences, since case count reporting by canton began mid-May 2020.}  
\label{fig:downsampling_representativeness}
\end{figure}

\begin{figure}[h!]
\centering
\includegraphics[width = 0.5\linewidth]{figures/chain_size_dist.png}
\caption{Size distribution of estimated introductions.}  
\label{fig:chain_size_dist}
\end{figure}

\begin{figure}[h!]
\centering
\includegraphics[width = 0.5\linewidth]{figures/chain_longevity_matrix.png}
\caption{Heatmap of the number of newly sampled introductions in Switzerland each month (diagonal entries) and the number continuing to persisting into each following month (off-diagonal entries). Introduction are counted once in the month they are first sampled (``Month of first sampling'') and one every following month (``Month of ongoing sampling'') until the date of the latest sample. The ranges are between two point estimates generated assuming either few or many introductions.}  
\label{fig:chain-longevity-matrix}
\end{figure}

\begin{figure}[h!]
\centering
\begin{subfigure}[b]{\textwidth}
\centering
\includegraphics[width = 11.4cm]{figures/fig_SX_sensitivity_subsampling.png}
\caption{Estimated introductions by percent (or, n=number) of Swiss confirmed cases analyzed. No. = number. Each box plot is a summary over 50 random sub-samples.}  
\end{subfigure}

\begin{subfigure}[b]{0.9\textwidth}
\centering
\includegraphics[width = 11.4cm]{figures/fig_SX_sensitivity_context_set_size.png}
\caption{Summary statistics about introductions by ratio of foreign context to focal Swiss sequences.}  
\end{subfigure}

\begin{subfigure}[b]{0.9\textwidth}
\centering
\includegraphics[width = 11.4cm]{figures/fig_SX_sensitivity_chain_defn.png}
\caption{Summary statistics about introductions and pipeline runtime by different heuristic definitions of an introduction.}  
\end{subfigure}
\caption{Sensitivity analyses for the definition of an introduction. (a) shows sensitivity to number of focal sequences analyzed, (b) shows sensitivity to the ratio of foreign context to focal sequences analyzed, and (c) shows sensitivity to the heuristic thresholds used to define an introduction given a tree. All statistics were generated under two different polytomy assumptions giving rise to either many or few introductions.}
\label{fig:sensitivity_figs}
\end{figure}

% \begin{landscape}
\begin{figure}[h!]
\centering
\begin{subfigure}[b]{\textwidth}
\includegraphics[width=\linewidth]{figures/CHE_sampProp.pdf}
\caption{Sampling probability in Switzerland.}
\end{subfigure}
\begin{subfigure}[b]{\textwidth}
\includegraphics[width=\linewidth]{figures/CHE_Re.pdf}
\caption{Effective reproductive number in Switzerland.}
\end{subfigure}
\begin{subfigure}[b]{\textwidth}
\includegraphics[width=\linewidth]{figures/NZL_Re.pdf}
\caption{Effective reproductive number in New Zealand.}
\end{subfigure}
\caption{Phylodynamic estimates for  (a) the sampling probability in Switzerland and the time-varying effective reproductive number $R_e$ in Switzerland (b) and New Zealand (c). $R_e$ estimates are overlaid on estimates generated from case count data \cite{huisman_re_preprint} in grey. Additionally, $R_e$ estimates from the models with a damping factor (pink) are the ``baseline'' $R_e$ before introduction-specific damping (i.e. before application of a damping factor once introductions are older than 2-days post sampling).}  
\label{fig:ReSampProbResults}
\end{figure}
% \end{landscape}
% Remember Re plot is now background Re (without contact tracing) and no longer the actual average Re

\begin{figure}[h!]
\centering
\begin{subfigure}[b]{0.7\textwidth}
\includegraphics[width=\linewidth]{figures/CHE_contact_tracing_factor.pdf}
\caption{Damping factor in Switzerland.}
\end{subfigure}
\begin{subfigure}[b]{0.7\textwidth}
\includegraphics[width=\linewidth]{figures/NZL_contact_tracing_factor.pdf}
\caption{Damping factor in New Zealand.}
\end{subfigure}
\caption{Phylodynamic estimates for the damping factor in Switzerland and New Zealand in different time periods, conditioned on introductions defined using different polytomy assumptions and using different priors on the sampling probability.}  
\label{fig:DampingFactorResults}
\end{figure}
\newpage

\begin{landscape}
\begin{figure}[h!]
\centering
\begin{subfigure}[b]{0.65\textwidth}
\centering
\includegraphics[width=0.75\linewidth]{figures/Re_skyline.max_chains.sampUB1.0.ctEst1.summary_trees.pdf}
\caption{Summary trees for introductions defined assuming many introductions.}
\end{subfigure}
\begin{subfigure}[b]{0.65\textwidth}
\centering
\includegraphics[width=0.75\linewidth]{figures/Re_skyline.min_chains.sampUB1.0.ctEst1.summary_trees.pdf}
\caption{Summary trees for introductions defined assuming few introductions.}
\end{subfigure}
\caption{Summary trees from the phylodynamic analysis for Switzerland. (a) and (b) show trees for introductions defined using different polytomy assumptions. The 50th and 95th percentile largest introductions among those first sampled each month are shown. The three color regions represent the Spring (green), Summer (orange) and Fall (blue) periods. The vertical dashed line shows the date at which the transmission rate can slow for each introduction - two days after the first sample date. The red bars show the 95\% highest posterior density uncertainty in node dates.}  
\label{fig:logged-chains}
\end{figure}
\end{landscape}
\newpage

\section{Supporting information tables}

\begin{table}[h!]
% \centering
\caption{Top 20 largest SARS-CoV-2 sequencing data contributors to GISAID in 2020 by submitting lab.}
\label{tab:seq-contributions}
\begin{tabular}{lll}
\hline
Submitting lab & Countries represented (ISO codes) & Number of sequences \\
\hline
Wellcome Sanger Institute for the COVID-19 Genomics UK (COG-UK) Consortium & GBR & 96441 \\ 
COVID-19 Genomics UK (COG-UK) Consortium & GBR & 71371 \\ 
Albertsen Lab, Department of Chemistry and Bioscience, Aalborg University, Denmark & DNK & 27936 \\ 
Houston Methodist Hospital & USA & 27409 \\ 
Pathogen Genomics Center, National Institute of Infectious Diseases & JPN; MMR & 19708 \\ 
Department of Biosystems Science and Engineering, ETH Zürich & CHE & 11357 \\ 
MDU-PHL & AUS; TLS & 10459 \\ 
TGen North & USA & 9491 \\ 
Wyoming Public Health Laboratory & USA & 9172 \\ 
Aalborg University & DNK & 8439 \\ 
SeqCOVID-SPAIN consortium/IBV(CSIC) & ESP & 8279 \\ 
Chan-Zuckerberg Biohub & USA & 7803 \\ 
BCCDC Public Health Laboratory & CAN & 7646 \\ 
Laboratoire de santé publique du Québec & CAN & 6914 \\ 
Andersen lab at Scripps Research & JOR; MEX; USA & 6258 \\ 
Utah Public Health Laboratory & USA & 5925 \\ 
MEPHI, Aix Marseille University & FRA & 5617 \\ 
Respiratory Virus Unit, Microbiology Services Colindale, Public Health England & GBR; UKR & 5142 \\ 
deCODE genetics & ISL & 5005 \\ 
Erasmus Medical Center & BEL; BHR; LUX; NLD; SUR & 4594 \\
\hline
\end{tabular}
\end{table}

\begin{table}[h!]
% \centering
\caption{Sampling proportion change-points for the phylodynamic analysis on Swiss data. The sampling proportion was modelled as a piece-wise-constant function in time, with the following change-points motivated by major shifts in the testing regime or genome sequencing intensity in Switzerland.}
\label{tab:CHE_samp_prob_change_dates}
\begin{tabular}{ll}
\hline
Start date & Description \\
\hline
23. Apr 2020 & All symptomatic individuals can get tested \\
25. Jun 2020 & Government pays for tests for symptomatic individuals \\
14. Sept 2020 & Genome sampling $<<$ 5\% of confirmed cases \\
28. Sept 2020 & Number of tests conducted and \% positivity dramatically increase, genome sampling also increases \\
19. Oct 2020 & Genome sampling $<<$ 5\% of confirmed cases again \\
11. Nov 2020 & Genome sampling improves again \\
\hline
\end{tabular}
\end{table}

\begin{table}[h!]
\caption{Contingency table for singleton introductions and transmission chains by time period assuming many (right) and few (left)  introductions.}
\label{tab:lockdown-contingency}
\input{tables/max_chains_contingency}
\input{tables/min_chains_contingency}
\end{table}

\input{tables/lineage_data_summary.tex}


\printbibliography{}

\end{document}
