\documentclass[9pt,twoside,lineno]{pnas-new}
% Use the lineno option to display guide line numbers if required.

\templatetype{pnasresearcharticle}
\usepackage{longtable}
\usepackage{array,ragged2e} % for nicer justification of table entries with "P"
\newcolumntype{P}[1]{>{\RaggedRight\arraybackslash}p{#1}}
\usepackage{caption} % for subfigures
\usepackage{subcaption}
\usepackage{lscape} % for turning one page to landscape

\let\thefigureWithoutS\thefigure %% <- store old definition
\renewcommand\thefigure{S\thefigureWithoutS}

\usepackage[backend=bibtex,style=nature]{biblatex}
\bibliography{references}

\title{Supporting information}
\author{}
% \correspondingauthor{Corresponding Author name.\\E-mail: author.two@email.com}

\begin{document}

\maketitle 

\tableofcontents

\newpage

\section{Materials and methods}
\subsection{Genome sequencing and quality filtering}
Most of the Swiss SARS-CoV-2 genomes analyzed until 31. Dec 2020 were generated by the Swiss SARS-CoV-2 Sequencing Consortium \cite{s3c-webpage}. Here, we briefly describe the swab-to-sequence process for these samples. RNA extracts from qPRC-positive patient naso- or oral-pharangeal swabs were provided by Viollier AG, a Swiss medical diagnostics company. RNA extraction was done with either the Abbott m2000sp or Seegene STARMag 96x4 Universal Cartridge RNA extraction kit. Extracts were then  transferred to the Genomics Facility Basel or the Functional Genomics Center Zurich for whole-genome sequencing. Both centers used the ARCTIC v3 primer scheme \cite{Quick2017, ARCTICNetwork} to generate tiled, approximately 400bp-long amplicons. Library preparation was done with the New England Biolabs (NEB) library preparation kit. Libraries were sequenced on Illumina MiSeq or NovaSeq machines, resulting in 2 x 251 base reads. Bioinformatics was done using V-pipe \cite{v-pipe-bioinformatics}, including read trimming and filtering with PRINSEQ \cite{Schmieder2011}, alignment to GenBank accession MN908947 \cite{Wu2020} with bwa \cite{Li2009}, and consensus base calling. For consensus base calling, positions with $<$5x coverage are masked with ``N'', positions with $>$5\% and $>$2 reads supporting a minor base are called with IUPAC ambiguity codes, and positions with $>$50\% reads supporting a deletion are called with ``-''. We rejected samples with $<$20,000 non-N bases. The consensus sequences we generated have been made publicly available on both GISAID (submitting lab: Department of Biosystems Science and Engineering, ETH ZÃ¼rich) and European Nucleotide Archive (ENA) (study: PRJEB38472).

We supplemented our Swiss data with other Swiss sequences and foreign sequences available via GISAID (accessed 31. May 2021). From the full set of sequences available on GISAID, we removed from consideration non-human samples, samples $<$ 27000 bases long, and samples flagged by the Nextclade tool \cite{Aksamentov} for one of the following reasons: suspiciously clustered SNPs (QC SNP clusters status metric not ``good''; $>=$ 6 mutations in 100 bases), too many private mutations (QC private mutations status metric not ``good''; $>=$ 10 mutations from the nearest tree node), or overall bad quality (Nextclade QC overall status ``bad''). We aligned the sequences to the reference genome MN908947.3 using MAFFT \cite{katoh_mafft:_2002}. Finally, we followed the Nextstrain pipeline's recommendation to mask the first 100 and last 50 sites of the alignment \cite{Nextstraina} since the start and end of SARS-CoV-2 sequences are prone to sequencing error \cite{DeMaio2020}.

\subsection{Sampling procedure}

From the quality-filtered alignment of GISAID sequences, we selected a focal set of sequences from Switzerland and a context set of sequences from abroad. 

% describe representativeness of Swiss dataset here too?
For the focal sequences from Switzerland, we aimed to select a spatially and temporally representative sample. Therefore, we down-sampled available sequences to \fpeval{\maxsamplingpercent}\% of confirmed case counts in each Swiss Canton each week between \mindate\ and \maxdate. Where there were not enough sequences available from a Canton in a week, we took all available sequences. To reduce the size of the alignments for phylogenetic analysis, we divided the focal Swiss set into Pango lineages \cite{Rambaut}, similar to \cite{DuPlessis2021}. Lineages composed of $>$ 50\% Swiss sequences were aggregated into their parent lineage(s) until $<=$ 50\% were Swiss. This aims to ensure that each analyzed lineage originated outside of Switzerland.

% describe nextstrain priority protocol for sim dataset
For the contextual sequences from abroad, we aimed to select the most genetically similar sequences to focal Swiss sequences. This set should help distinguish between SARS-CoV-2 variants unique to Switzerland (likely within-Switzerland transmission) and variants also circulating abroad (possibly recent introductions). To select this set, we applied the Nextstrain priority script \cite{Nextstrainteam} to rank sequences from abroad by their genetic similarity to Swiss sequences in each lineage alignment. Then, we selected \similaritycontextscalefactor\ times as many  context sequences as focal Swiss sequences for each analyzed lineage. 

% summarize the sample set
In the supporting information, we show sensitivity analyses for these choices. We tested different numbers of focal Swiss sequences and different ratios of contextual to focal sequences, with the conclusion that the most variation is still due to how we define an introduction (see ``Identifying introductions'', below). The final dataset used for analysis includes \nfocalsamples\ focal sequences from Switzerland and \nsimcontext\ genetically similar contextual sequences from abroad.

\subsection{Timetree generation}
We estimated the maximum likelihood phylogeny for each lineage alignment using IQ-TREE \cite{Nguyen2014} under an HKY substitution model \cite{Hasegawa1985} with empirical base frequencies and 4 gamma rate categories. We then rooted each phylogeny with \outgroupgisaidepiisls\ as an outgroup and estimated branch lengths in time units using least-squares dating (LSD) \cite{To2016} with a strict molecular clock and a minimum mutation rate of 8x10-4 substitutions per site per year. We additionally assumed the root date to be between 15. Nov. and 24. Dec. 2019 (roughly in line with estimates provided by Nextstrain \cite{Nextstrainteam}) and set the minimum branch length to zero. Sequences that violated the strict clock assumption (z-score threshold > 3) were removed and near-zero branches ($<$1.7x10-5 substitutions per site per year) were collapsed into polytomies, reflecting the fact that the sequence data is not sufficient to resolve the ordering of these transmission events. Given the root date constraints, the mutation rate conformed to the lower bound of 8x10-4 with extremely narrow confidence intervals. Confidence intervals for node dates were generated in LSD by re- sampling branch lengths 100 times under a lognormal relaxed clock model with standard deviation 0.4.

\subsection{Identifying introductions}
We ``picked'' putative Swiss transmission chains (collections of two or more genomic sequences resulting from within-Switzerland transmissions) off of each Pango lineage tree according to the following criteria: at least 2 Swiss sequences are part of a clade in the tree and the subtree spanned by these Swiss sequences is monophyletic upon removing (a) up to 3 export events where (b) only one export event may occur along each internal branch. Exports are defined to be clades containing non-Swiss sequences. We chose a conservative value for criterion (b) while still allowing some export events and note that the number of inferred transmission chains is robust to different values for criterion (a) given criterion (b) (Fig. \ref{fig:sensitivity_figs}c). For a sensitivity analysis on the importance of the specific values of criteria (a) and (b) see the supporting text.

We refer to any Swiss sequence not falling into a Swiss transmission chain as a Swiss singleton. We assume each singleton and each transmission chain represent an independent introduction of SARS-CoV-2 into Switzerland; together these are called introductions.

Finally, we repeated this picking procedure twice for each lineage tree, interpreting polytomies in two different ways. Once, we split Swiss clades descending from polytomies into independent introductions unless the polytomy only had a single non-Swiss descendent (see criterion (b)). Alternatively, we aggregated all Swiss clades descending from polytomies into a single introduction. These procedures are conceptually similar to the ACCTRANS (accelerated transformations) and DELTRANS (delayed transformations) methods for assigning character transformations when multiple scenarios are equally parsimonious \cite{Miyakawa2004}. In summary, we identify introductions twice, generating estimates that represent two plausible extremes of  many introductions and few introductions.

\subsection{Quantifying the effect of border closures on introductions}
We wanted to understand if the Swiss border closure on 13 Mar. 2020 changed introduction dynamics. Our null model is that the number of weekly introductions before border closure, estimated from the genome data as described above, is a linear function of weekly SARS-CoV-2 cases in surrounding countries. Introductions are dated by the first detected case, whereas case counts (taken from the ECDC \cite{ECDC}) are dated by case reporting dates. We considered an average delay between these variables up to 18 days. We also considered two options for ``surrounding countries'', either all the non-Swiss European countries in the ECDC dataset or just Switzerland's neighboring countries Italy, France, Germany, and Austria. We fit these models separately to the weekly many and few introductions estimates before 13 Mar. 2020 and selected the best-fit models (lowest root mean squared error). For both estimates, the neighboring-countries-only incidence data better predicted introductions. The best-fit delay was \casesdelaymax days for the many introductions estimates compared to \casesdelaymin days for the few introductions estimates. Figure \ref{fig:introductions_and_persistence_v2} shows these best-fit models and projections if these dynamics had continued after 13. Mar.

\subsection{Quantifying the effect of lockdown conditions on persistence}
We also wanted to understand if the Swiss lockdown between 17 Mar. and 27 Apr. 2020 changed the persistence of introductions. Our null model is that introductions circulating on any given day persist equally long, regardless of the day. In other words, introductions die out (are no longer sampled) according to some delay distribution that is constant through time. For each date, we calculated the median and inter-quartile range of the time from that date to the last sample for all introductions persisting on that date. Singleton introductions trivially persist for 1 day. Figure \ref{fig:introductions_and_persistence_v2} shows how the persistence of the estimated introductions changes around the dates of the Swiss lockdown from 17. Mar to 27. Apr.

\subsection*{Phylodynamic analysis}

After identifying introductions, we performed phylodynamic inference on them using the BDSKY (birth-death skyline) method \cite{stadler2013birth} in BEAST2 \cite{Bouckaert2019}. To avoid model mis-specification due to the more transmissible alpha variant, we analyzed data only through 30 Nov.~2020.  The inference relies on two main models: a nucleotide substitution model describing an evolutionary process and a population dynamical model describing a transmission and sampling process. 

For the nucleotide substitution model, we assumed an HKY \cite{Hasegawa1985} model with 4 Gamma rate categories to account for site-to-site rate heterogeneity \cite{Yang1994}. We used the default priors for kappa and the scale factor of the Gamma distribution. We assumed a strict clock with rate $8\times 10^{-4}$ substitutions per site per year.

For the population dynamic model, we used BDSKY \cite{stadler2013birth}. In BDSKY, the identified introductions are the result of a birth-death with sampling process parameterized by an effective reproductive number, a becoming-uninfectious rate, and a sampling probability. As in \cite{Muller2020}, we inferred these population dynamical parameters jointly from the different introductions. More concretely, each introduction is assumed to result from an independent birth-death process having its own start time, but sharing all other parameters with the processes associated with the other introductions. 
Further, we used a uniform prior on the time of origin for each introduction, from 01. Jan until the end of the sampling period. This means our prior expectation is a uniform rate of introductions through time. We fixed the become-uninfectious rate to 36.5 per year, which corresponds to an average of 10 days to becoming uninfectious. We allowed the effective reproductive number to vary week-to-week. We applied an Ornstein-Uhlenbeck smoothing prior to the logarithm of this parameter, defined such that the stationary distribution of the process was $\text{LogNormal}(0.8, 0.5)$  and with an $\text{Exp}(1)$ hyperprior on the relaxation parameter of the process. This prior constrains both the absolute reproductive number and the relative sizes of the change between weeks. Finally, we allowed the sampling probability to vary when Swiss testing or genome sampling regimes changed significantly (Table S1). For our main analysis, we applied a $\text{LogUniform}(10^{-4}, 1)$ prior on the sampling probability. As a sensitivity analysis, we also tried a $\text{LogUniform}(10^{-4}, 0.05)$ prior since we upper-bounded our sampling to 5\% of confirmed cases each week. See the supporting text for more information.

Then, we added an additional ``contact tracing'' factor to the model. This factor is a multiplicative damping of the effective reproductive number applied to each introduction from 2 days after the first sample date until the introduction dies out. Since we hypothesized contact tracing was not functioning as well during periods of high case numbers, we estimated a separate value in each of three periods: before 15. Jun 2020 (Spring), 15 Jun.~to 30 Sept.~2020 (Summer), and 30. Sept to \maxdate\ (Fall). We used the same spike and slab prior for the damping factor in each period, with an inclusion probability of 0.5 and a uniform prior if included between 0 and 1. 

The XML files for the phylodynamic analysis are available on the project GitHub repository at TODO.

\section{Supporting text}

% \subsection*{Subhead}
% Type or paste text here. This should be additional explanatory text such as an extended technical description of results, full details of mathematical models, etc.   
\subsection{Sensitivity analyses for identifying introductions}
Here we describe different sensitivity analyses we performed for the definition of an introduction. 

\subsubsection{Number of focal Swiss sequences analyzed}
To assess whether the number of identified introductions saturates as we add more sequences, we did a sub-sampling analysis. We sub-sampled the Swiss genome sequences used in our main analysis from 5\% of confirmed cases down to 1\% of confirmed cases. For each new sub-sampling value, we calculated the number of introductions (singletons and transmission chains) that we would have identified given the sub-sampled sequence set. Figure \ref{fig:sensitivity_figs}a shows that as we add sequences, we do not reach saturation. Therefore, if we were to include even more sequences, we would almost certainly identify more introductions into Switzerland.

\subsubsection{Ratio of foreign context to focal Swiss sequences analyzed}
Next, we assessed how summary statistics about identified introductions and the runtime of our pipeline changed as we added more and more foreign context sequences. We kept the number of focal Swiss sequences constant at 5\% of confirmed cases each week and added foreign context sequences at a 1:1, 2:1, and 3:1 ratio to focal Swiss sequences. Figure \ref{fig:sensitivity_figs}b shows that as we add foreign context sequences, we identify more numerous, smaller introductions. However, the greatest differences come from the different assumptions about how to resolve polytomies (few vs. many introductions), not the ratio of foreign context to focal Swiss sequences. Therefore, we chose to present results using the 2:1 ratio to balance speed (smaller dataset = faster tree search convergence) and information content (larger dataset = more precise estimates of introduction dates).

\subsubsection{Criteria for identifying introductions}
After deciding on a ratio of foreign context to focal Swiss sequences, we assessed how summary statistics about identified introductions changed depending on the precise heuristic definition of an introduction. Given phylogenetic trees with polytomies, we picked transmission chains off of these trees as described in the main text methods. Here, we varied (a) the number of export events allowed in each transmission chain and (b) the maximum number of export events allowed to occur along each single internal branch. Figure \ref{fig:sensitivity_figs}c shows that increasing (a) yields fewer, larger transmission chains. Increasing (b) for each level of (a) has a much smaller effect. We note that again, the greatest differences come from the different assumptions about how to resolve polytomies. Therefore, we chose to present results using a transmission chain definition based on (a) maximum 3 exported lineages and (b) maximum one consecutive export on each internal branch. This allows for some exports from Swiss transmission chains but not arbitrarily many. 

\subsection{Sensitivity analyses for phylodynamic modelling}
Here we describe a sensitivity analyses and some example intermediate outputs from our phylodynamic analysis.

\subsubsection{Sampling proportion prior}
We wanted to check that our phylodynamic estimates of a transmission damping factor are robust to our prior on the sampling proportion. Therefore, we repeated our analyses using two different priors on the sampling proportion. The first prior was a uniform distribution between 0 and 1. This broad prior allows the sampling proportion to assume any plausible value. The second prior was a uniform distribution between 0 and an upper bound equal to the number of genome sequences analyzed in a week divided by the number of confirmed cases that week. This narrower prior assumes that there are at least as many cases each week as confirmed cases. Figure \ref{fig:ReSampProbResults}a shows that in Switzerland, the estimated sampling proportion in late Fall 2020 varies greatly depending on the prior. A drop in SARS-CoV-2 diversity in Switzerland during this period might explain why the inference under the broader sampling prior estimates a proportion corresponding to fewer individuals than we know were infected during this time. Figure \ref{fig:ReSampProbResults}b shows that the effective reproductive number estimates in Fall 2020 for Switzerland more closely match estimates based on line-list data when the sampling proportion is treated as a fitting parameter, i.e. under the first, broad prior. Therefore, we report results under this prior in the main text. In Figure \ref{fig:DampingFactorResults}a, we show that the damping factor results are qualitatively similar between the two sampling proportion priors. 

\subsubsection{Logged trees}
Finally, we logged phylogenetic trees for a few introductions. These trees were sampled by the Markov chains in the phylodynamic analyses. Note that the damping factor results are jointly inferred from all the branching events across introductions in each time period. But since logging all introductions would yield huge log files, we logged just a few individual introductions under each set of model assumptions for visual inspection. For each set of model assumptions and each month, we logged trees for the 50th and 95th percentile largest introductions that were first sampled that month and eventually yielded > 2 samples. Figure \ref{fig:logged-chains} shows as an example summary trees for these introductions under the phylodynamic analysis for Switzerland with an un-bounded sampling proportion prior and a homogeneous population (no class of individuals with more contacts).

\newpage

\section{Supporting information figures}

\begin{figure}[h!]
\centering
\includegraphics[width = 0.8\linewidth]{figures/CHE_downsampling.png}
\caption{Spatio-temporal representativeness of analyzed genome sequences. The mirror y-axis aims to contrast temporal trends in confirmed cases (red) versus analyzed sequences (blue); all values are positive counts. Facet titles are standard abbreviations for Swiss cantons. 'CHE' represents Swiss-wide cases and sequences, since case count reporting by canton began mid-May 2020.}  
\label{fig:downsampling_representativeness}
\end{figure}

\begin{figure}[h!]
\centering
\includegraphics[width = 0.5\linewidth]{figures/chain_size_dist.png}
\caption{Size distribution of estimated introductions.}  
\label{fig:chain_size_dist}
\end{figure}

\begin{figure}[h!]
\centering
\includegraphics[width = 0.5\linewidth]{figures/chain_longevity_matrix.png}
\caption{Heatmap of the number of newly sampled introductions in Switzerland each month (diagonal entries) and the number continuing to persisting into each following month (off-diagonal entries). Introduction are counted once in the month they are first sampled (``Month of first sampling'') and one every following month (``Month of ongoing sampling'') until the date of the latest sample. The ranges are between two point estimates generated assuming either few or many introductions.}  
\label{fig:chain-longevity-matrix}
\end{figure}

\begin{figure}[h!]
\centering
\begin{subfigure}[b]{\textwidth}
\centering
\includegraphics[width = 11.4cm]{figures/fig_SX_sensitivity_subsampling.png}
\caption{Estimated introductions by percent (or, n=number) of Swiss confirmed cases analyzed. No. = number. Each box plot is a summary over 50 random sub-samples.}  
\end{subfigure}

\begin{subfigure}[b]{0.9\textwidth}
\centering
\includegraphics[width = 11.4cm]{figures/fig_SX_sensitivity_context_set_size.png}
\caption{Summary statistics about introductions by ratio of foreign context to focal Swiss sequences.}  
\end{subfigure}

\begin{subfigure}[b]{0.9\textwidth}
\centering
\includegraphics[width = 11.4cm]{figures/fig_SX_sensitivity_chain_defn.png}
\caption{Summary statistics about introductions and pipeline runtime by different heuristic definitions of an introduction.}  
\end{subfigure}
\caption{Sensitivity analyses for the definition of an introduction. (a) shows sensitivity to number of focal sequences analyzed, (b) shows sensitivity to the ratio of foreign context to focal sequences analyzed, and (c) shows sensitivity to the heuristic thresholds used to define an introduction given a tree. All statistics were generated under two different polytomy assumptions giving rise to either many or few introductions.}
\label{fig:sensitivity_figs}
\end{figure}

\begin{figure}[h!]
    \centering
    \includegraphics[width=\linewidth]{figures/introductions_and_persistence_v2.png}
    \caption{Introductions into Switzerland and their persistence around the time of the Swiss border closures and partial lockdown. (A) compares weekly introductions estimated from the genome data (bars) with predicted introductions based on case counts in Italy, France, Germany, and Austria (shaded area). The shaded area was fit to the genome-based estimates prior to the dashed line on 13 Mar 2020, the date of Swiss border closures, and the model fit is extrapolated after this date. (B) shows how the persistence of estimated introductions changed around the time of the Swiss lockdown, which is also represented by dashed lines. Persistence is shown as the median time from each date until introductions circulating on that date are last sampled. The shaded region shows the interquartile range and the two different colors are for estimates generated from the two different plausible sets of introductions, few and many.}
    \label{fig:introductions_and_persistence_v2}
\end{figure}

% \begin{landscape}
\begin{figure}[h!]
\centering
\begin{subfigure}[b]{\textwidth}
\includegraphics[width=\linewidth]{figures/CHE_sampProp.pdf}
\caption{Sampling probability in Switzerland.}
\end{subfigure}
\begin{subfigure}[b]{\textwidth}
\includegraphics[width=\linewidth]{figures/CHE_Re.pdf}
\caption{Effective reproductive number in Switzerland.}
\end{subfigure}
\begin{subfigure}[b]{\textwidth}
\includegraphics[width=\linewidth]{figures/NZL_Re.pdf}
\caption{Effective reproductive number in New Zealand.}
\end{subfigure}
\caption{Phylodynamic estimates for  (a) the sampling probability in Switzerland and the time-varying effective reproductive number $R_e$ in Switzerland (b) and New Zealand (c). $R_e$ estimates are overlaid on estimates generated from case count data \cite{huisman_re_preprint} in grey. Additionally, $R_e$ estimates from the models with a damping factor (pink) are the ``baseline'' $R_e$ before introduction-specific damping (i.e. before application of a damping factor once introductions are older than 2-days post sampling).}  
\label{fig:ReSampProbResults}
\end{figure}
% \end{landscape}
% Remember Re plot is now background Re (without contact tracing) and no longer the actual average Re

\begin{figure}[h!]
\centering
\begin{subfigure}[b]{0.7\textwidth}
\includegraphics[width=\linewidth]{figures/CHE_contact_tracing_factor.pdf}
\caption{Damping factor in Switzerland.}
\end{subfigure}
\begin{subfigure}[b]{0.7\textwidth}
\includegraphics[width=\linewidth]{figures/NZL_contact_tracing_factor.pdf}
\caption{Damping factor in New Zealand.}
\end{subfigure}
\caption{Phylodynamic estimates for the damping factor in Switzerland and New Zealand in different time periods, conditioned on introductions defined using different polytomy assumptions and using different priors on the sampling probability.}  
\label{fig:DampingFactorResults}
\end{figure}
\newpage

\begin{landscape}
\begin{figure}[h!]
\centering
\begin{subfigure}[b]{0.65\textwidth}
\centering
\includegraphics[width=0.75\linewidth]{figures/Re_skyline.max_chains.sampUB1.0.ctEst1.summary_trees.pdf}
\caption{Summary trees for introductions defined assuming many introductions.}
\end{subfigure}
\begin{subfigure}[b]{0.65\textwidth}
\centering
\includegraphics[width=0.75\linewidth]{figures/Re_skyline.min_chains.sampUB1.0.ctEst1.summary_trees.pdf}
\caption{Summary trees for introductions defined assuming few introductions.}
\end{subfigure}
\caption{Summary trees from the phylodynamic analysis for Switzerland. (a) and (b) show trees for introductions defined using different polytomy assumptions. The 50th and 95th percentile largest introductions among those first sampled each month are shown. The three color regions represent the Spring (green), Summer (orange) and Fall (blue) periods. The vertical dashed line shows the date at which the transmission rate can slow for each introduction - two days after the first sample date. The red bars show the 95\% highest posterior density uncertainty in node dates.}  
\label{fig:logged-chains}
\end{figure}
\end{landscape}
\newpage

\section{Supporting information tables}

\begin{table}[h!]
% \centering
\caption{Top 20 largest SARS-CoV-2 sequencing data contributors to GISAID in 2020 by submitting lab.}
\label{tab:seq-contributions}
\begin{tabular}{lll}
\hline
Submitting lab & Countries represented (ISO codes) & Number of sequences \\
\hline
Wellcome Sanger Institute for the COVID-19 Genomics UK (COG-UK) Consortium & GBR & 96441 \\ 
COVID-19 Genomics UK (COG-UK) Consortium & GBR & 71371 \\ 
Albertsen Lab, Department of Chemistry and Bioscience, Aalborg University, Denmark & DNK & 27936 \\ 
Houston Methodist Hospital & USA & 27409 \\ 
Pathogen Genomics Center, National Institute of Infectious Diseases & JPN; MMR & 19708 \\ 
Department of Biosystems Science and Engineering, ETH ZÃ¼rich & CHE & 11357 \\ 
MDU-PHL & AUS; TLS & 10459 \\ 
TGen North & USA & 9491 \\ 
Wyoming Public Health Laboratory & USA & 9172 \\ 
Aalborg University & DNK & 8439 \\ 
SeqCOVID-SPAIN consortium/IBV(CSIC) & ESP & 8279 \\ 
Chan-Zuckerberg Biohub & USA & 7803 \\ 
BCCDC Public Health Laboratory & CAN & 7646 \\ 
Laboratoire de santÃ© publique du QuÃ©bec & CAN & 6914 \\ 
Andersen lab at Scripps Research & JOR; MEX; USA & 6258 \\ 
Utah Public Health Laboratory & USA & 5925 \\ 
MEPHI, Aix Marseille University & FRA & 5617 \\ 
Respiratory Virus Unit, Microbiology Services Colindale, Public Health England & GBR; UKR & 5142 \\ 
deCODE genetics & ISL & 5005 \\ 
Erasmus Medical Center & BEL; BHR; LUX; NLD; SUR & 4594 \\
\hline
\end{tabular}
\end{table}

\begin{table}[h!]
% \centering
\caption{Sampling proportion change-points for the phylodynamic analysis on Swiss data. The sampling proportion was modelled as a piece-wise-constant function in time, with the following change-points motivated by major shifts in the testing regime or genome sequencing intensity in Switzerland.}
\label{tab:CHE_samp_prob_change_dates}
\begin{tabular}{ll}
\hline
Start date & Description \\
\hline
23. Apr 2020 & All symptomatic individuals can get tested \\
25. Jun 2020 & Government pays for tests for symptomatic individuals \\
14. Sept 2020 & Genome sampling $<<$ 5\% of confirmed cases \\
28. Sept 2020 & Number of tests conducted and \% positivity dramatically increase, genome sampling also increases \\
19. Oct 2020 & Genome sampling $<<$ 5\% of confirmed cases again \\
11. Nov 2020 & Genome sampling improves again \\
\hline
\end{tabular}
\end{table}

\begin{table}[h!]
\caption{Contingency table for singleton introductions and transmission chains by time period assuming many (right) and few (left)  introductions.}
\label{tab:lockdown-contingency}
\input{tables/max_chains_contingency}
\input{tables/min_chains_contingency}
\end{table}

\input{tables/lineage_data_summary.tex}


\printbibliography{}

\end{document}
