\documentclass[9pt,twoside,lineno]{pnas-new}
% Use the lineno option to display guide line numbers if required.

\templatetype{pnassupportinginfo}
\usepackage{longtable}
% \usepackage{biblatex}
% \addbibresource{references.bib}

\title{Your main manuscript title}
\author{Author1, Author2 and Author3 (complete author list)}
\correspondingauthor{Corresponding Author name.\\E-mail: author.two@email.com}

% \input{manuscript/chain_summary_variables}

\begin{document}

%% Comment out or remove this line before generating final copy for submission; this will also remove the warning re: "Consecutive odd pages found".
% \instructionspage  

\maketitle 

%% Adds the main heading for the SI text. Comment out this line if you do not have any supporting information text.
\SItext


% \subsection*{Subhead}
% Type or paste text here. This should be additional explanatory text such as an extended technical description of results, full details of mathematical models, etc.   
\section{Sensitivity analyses}

In this section, we describe our decision-making process to choose parameters for our phylogenetic analysis and report on the sensitivity of our results to these choices. 

\subsection{Down-sampling Swiss sequences}
We aim to include Swiss sequences that are spatio-temporally representative of the Swiss epidemic. Therefore, we down-sample available Swiss sequences proportional to confirmed case counts at the regional (cantonal) level. To assess whether the number of transmission chains saturates as we add more and more sequences, we did a saturation analysis. TODO: add figure and discuss implications.

\subsection{Ratio of foreign to Swiss sequences analyzed}
We investigated the sensitivity of transmission chain summary statistics to different ratios of genetically similar context sequences from abroad to focal Swiss sequences. Figure \ref{fig:sensitivity_context_set_size} shows that increasing the ratio of context to focal sequences from 1:1 to 3:1 increases the number of estimated transmission chains and decreases their mean size. However, the greatest differences in estimated transmission chains result from the different assumptions about how to resolve phylogenetic uncertainty (few vs. many introductions), not the ratio of foreign to Swiss sequences. Therefore, we chose the 2:1 ratio as a balance of speed (smaller dataset = faster tree search convergence) and information content (larger dataset = more precise estimates of introduction dates). We rely on the difference between the few and many introductions assumptions to represent uncertainty in the transmission chains.

\subsection{Criteria for identifying introductions}
After deciding on a ratio of foreign to Swiss sequences, we investigated the sensitivity of transmission chain summary statistics to our criteria for picking transmission chains. Figure \ref{fig:sensitivity_m_p} shows that increasing the maximum number of exported lineages decreases the number of transmission chains and increases their size. Increasing the maximum consecutive exports has a much smaller effect. But again, these differences are smaller than those resulting from the different assumptions about how to resolve phylogenetic uncertainty (few vs. many introductions). Therefore, we chose our criteria to be maximum 3 exported lineages and maximum one consecutive export to allow for some exports but not arbitrarily many. 

\subsection{Upper-bounding sampling proportion}
Genome-based estimates for the effective reproductive number in Switzerland are for the most part robust to the different transmission chain definitions and largely match estimates based on confirmed case counts (Figure \ref{fig:re}). One exception is in October 2020, when genome-based estimates are lower than confirmed case counts unless the sampling proportion is treated as a fitting parameter. A drop in genome diversity during this period might explain the discrepancy. 

\subsection{Including a ``super-spreader'' deme}
We checked whether our estimate of a transmission slow-down upon introduction sampling was robust to the inclusion of super-spreading individuals in the model. Therefore, we included a second deme in the analysis. Let $\pi_N$ be the fraction of normal individuals and $\pi_{SS}$ be the fraction of super-spreading individuals in the population. Super-spreading individuals transmit according to the master time-varying reproductive number multiplied by an additional positive factor $r_{SS}$. $\pi_N$, $\pi_{SS}$, and $r_{SS}$ are all inferred parameters. 

\section{Sanity checks}
From the phylodynamic analyses, we are primarily interested in estimating a transmission damping factor upon introductions being sampled. To generate this result, we are integrating over the actual introduction phylogenies using MCMC. However, as a sanity-check we logged the MCMC-sampled phylogenies for several introductions under each model. We chose which introductions to log based on selecting in each month the 50th and 75th percentile largest introductions that were first sampled that month and eventually yielded > 2 samples. We look at a summary visualization of the logged phylogenies to get an idea how much transmission happended before versus after the first sample and whether there is obvious signal in the phylogenies for a transmission slow-down after sampling.

% %%% Each figure should be on its own page
\begin{figure}
\centering
\includegraphics[width = 11.4cm]{figures/CHE_downsampling.png}
\caption{Spatio-temporal representativeness of analyzed genome sequences. The mirror y-axis aims to contrast temporal trends in confirmed cases (red) versus analyzed sequences (blue); all values are positive counts. Facet titles are standard abbreviations for Swiss cantons. 'CHE' represents Swiss-wide cases and sequences, since case count reporting by canton began mid-May 2020.}  
\label{fig:downsampling_representativeness}
\end{figure}

\begin{figure}
\centering
\includegraphics[width = 11.4cm]{figures/chain_size_dist.png}
\caption{Size distribution of estimated transmission chains.}  
\label{fig:chain_size_dist}
\end{figure}

\begin{figure}
\centering
\includegraphics[width = 11.4cm]{figures/fig_SX_sensitivity_subsampling.png}
\caption{Number of estimated introductions as a function of the fraction of Swiss confirmed cases analyzed.}  
\label{fig:sensitivity_downsampling}
\end{figure}

\begin{figure}
\centering
\includegraphics[width = 11.4cm]{figures/fig_SX_sensitivity_context_set_size.png}
\caption{Sensitivity of transmission chain summary statistics to different ratios of focal Swiss sequences to genetically similar foreign context sequences (1:1, 1:2, and 1:3).}  
\label{fig:sensitivity_context_set_size}
\end{figure}

\begin{figure}
\centering
\includegraphics[width = 11.4cm]{figures/fig_SX_sensitivity_chain_defn.png}
\caption{Sensitivity of transmission chain summary statistics to different definitions of a transmission chain.}  
\label{fig:sensitivity_m_p}
\end{figure}

\begin{figure}[tbhp]
\centering
\includegraphics[width=\linewidth]{figures/bdsky_2021-08-18/Re_CHE_1deme.pdf}
\vfill
\includegraphics[width=\linewidth]{figures/bdsky_2021-08-18/sampProp_CHE_1deme.pdf}
\caption{Phylodynamic estimates for the time-varying effective reproductive number (top) and sampling proportion (bottom) in Switzerland under the base model with a damping factor, with and without bounded sampling.}  
\label{fig:1DemeCHResults}
\end{figure}
% Remember Re plot is now background Re (without contact tracing) and no longer the actual average Re

\begin{figure}[tbhp]
\centering
\includegraphics[width=0.4\linewidth]{figures/bdsky_2021-08-18/CT_dampingProbs_CHE_1deme.pdf}
\includegraphics[width=0.4\linewidth]{figures/bdsky_2021-08-18/CT_dampingProbs_2deme.pdf}
\includegraphics[width=0.4\linewidth]{figures/bdsky_2021-08-18/CT_conditionedDamping_CHE_1deme.pdf}
\includegraphics[width=0.4\linewidth]{figures/bdsky_2021-08-18/CT_conditionedDamping_2deme.pdf}
\caption{Phylodynamic estimates for the damping factor in Switzerland. Top is inclusion probability for the damping factor, bottom is dampling factor estimate conditioned on inclusion. Left are inferences under the base model and right are inferences under the base model with a SS (super-spreading) deme.}  
\label{fig:1vs2demeDampingFactorResults}
\end{figure}
\newpage

\begin{figure}[tbhp]
\centering
\includegraphics[width=\linewidth]{figures/bdsky_2021-08-18/CT_dampingProbs_NZL.pdf}
\includegraphics[width=\linewidth]{figures/bdsky_2021-08-18/CT_conditionedDamping_NZL.pdf}
\caption{Phylodynamic estimates for the damping factor in New Zealand. Top is inclusion probability for the damping factor, bottom is dampling factor estimate conditioned on inclusion.}  
\label{fig:NZLDampingFactorResults}
\end{figure}
\newpage

\begin{table}[H]
\caption{Various phylodynamic models employed for the main analysis and sensitivity checks.}
\label{tab:phylo-models}
\begin{tabular}{cccccc}
Data set & Polytomy resolution & Damping factor (contact\_tracing) & Damping factor + SS Deme & Damping factor + Sampling bound \\
\midrule
Swiss & Many introductions (max) & Y & Y & Y \\
Swiss & Few introductions (min) & Y & Y & Y \\
New Zealand & Many introductions (max) & Y & & \\
New Zealand & Few introductions (min) & Y & & \\
\bottomrule
\end{tabular}
\newline
\addtabletext{Models with a ``Damping factor'' include a parameter representing a possible transmission slow-down upon sampling. Models with a ``SS deme'' include a second deme representing super-spreaders. Models with ``Sampling bound'' have the sampling proportion restricted to be less than the number of genomes divided by the number of confirmed cases.}
\end{table}

\newpage
\begin{table}[H]
\centering
\caption{Sampling proportion change-points in the phylodynamic analyses.}
\begin{tabular}{lp{6cm}rr}
Start date & Description \\
\midrule
23. Apr 2020 & All symptomatic individuals can get tested \\
25. Jun 2020 & Government pays for tests for symptomatic individuals \\
14. Sept 2020 & Genome sampling $<<$ 5\% of confirmed cases \\
28. Sept 2020 & Number of tests conducted and \% positivity dramatically increase, genome sampling also increases \\
19. Oct 2020 & Genome sampling $<<$ 5\% of confirmed cases again \\
11. Nov 2020 & Genome sampling improves again \\
\bottomrule
\end{tabular}
\end{table}

\newpage
\begin{table}[H]
\caption{Contingency table for singleton introductions and transmission chains by time period assuming many (right) and few (left)  introductions.}
\input{figures/max_chains_contingency}
\input{figures/min_chains_contingency}
\label{tab:lockdown-contingency}
\end{table}

\newpage
\input{figures/lineage_data_summary}

% %%% Add this line AFTER all your figures and tables
% \FloatBarrier

% \movie{Type legend for the movie here.}

% \movie{Type legend for the other movie here. Adding longer text to show what happens, to decide on alignment and/or indentations.}

% \movie{A third movie, just for kicks.}

% \dataset{dataset_one.txt}{Type or paste legend here.}

% \dataset{dataset_two.txt}{Type or paste legend here. Adding longer text to show what happens, to decide on alignment and/or indentations for multi-line or paragraph captions.}

\bibliography{references}

\end{document}
