---
title: "Test augur for ancestral state reconstruction"
author: "Sarah Nadeau"
date: "29/04/2021"
output: html_document
---

I left off wondering whether I want to/can run augur traits on the exact same tree.
And not settled on how to efficiently extract augur trait data from the pipeline's json output.

# Set up R environment
```{r, setup, include=FALSE}
knitr::opts_knit$set(root.dir = "/Users/nadeaus/Repos/cov-swiss-phylogenetics")
knitr::opts_chunk$set(root.dir = "/Users/nadeaus/Repos/cov-swiss-phylogenetics")
```
```{r}
source("../grapevine/database/R/utility.R")
source("../grapevine/utility_functions.R")
source("../grapevine/generate_figures/functions.R")
source("scripts/figures_shared_vars.R")
require(dplyr)
require(ggplot2)
db_connection = open_database_connection(config_file = "../grapevine/workdir/input/config.yml")
```

# Copy over trees from precision validation analysis
```{bash}
WORKDIR=jan-dec_-005_max_sampling_-25_travel_-5_sim_context-sf_100_travel-wt
mkdir -p /Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/validation/$WORKDIR

# scp nadeaus@euler.ethz.ch:/cluster/scratch/nadeaus/grapevine/$WORKDIR/tmp/iqtree/*.treefile \
# /Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/validation/$WORKDIR
scp nadeaus@euler.ethz.ch:/cluster/scratch/nadeaus/grapevine/$WORKDIR//*/B.1.1.7.fasta \
/Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/validation/$WORKDIR

# WORKDIR=jan-dec_-005_max_sampling_-25_travel_-5_sim_context-sf_111_travel-wt
# mkdir -p /Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/validation/$WORKDIR

# scp nadeaus@euler.ethz.ch:/cluster/scratch/nadeaus/grapevine/$WORKDIR/tmp/iqtree/*.treefile \
# /Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/validation/$WORKDIR
```

# Generate metadata for augur & weights file
tsv/csv table with equilibrium probabilities of discrete states
```{r}
workdir <- "results_all/validation/jan-dec_-005_max_sampling_-25_travel_-5_sim_context-sf_100_travel-wt"
metadata <- load_sample_metadata(
  workdir = workdir)
metadata_augur <- metadata %>%
  rename(name = tree_label, strain_prev = strain) %>%
  filter(!duplicated(x = name))  # remove duplicate entries corresponding to outgroup
write.csv(
  x = metadata_augur,
  file = paste(workdir, "output/augur/metadata_augur.csv", sep = "/"))
weights <- data.frame(
  state = unique(metadata$iso_country)
) # dropped this because it's not used by nextstrain? See: https://discussion.nextstrain.org/t/mugration-weights/101
```

```{bash}
LINEAGE=B.1.1.7
BASEDIR=/Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/validation
WORKDIR=jan-dec_-005_max_sampling_-25_travel_-5_sim_context-sf_100_travel-wt

mkdir -p $BASEDIR/$WORKDIR/output/augur
mkdir -p $BASEDIR/$WORKDIR/output/auspice

TREE=$BASEDIR/$WORKDIR/tmp/iqtree/${LINEAGE}.treefile  # newick tree
METADATA=$BASEDIR/$WORKDIR/output/augur/metadata_augur.csv
OUTPUT_NODE_DATA=$BASEDIR/$WORKDIR/output/augur/${LINEAGE}_augur_traits_iso_country.json
OUTPUT_AUSPICE=$BASEDIR/$WORKDIR/output/auspice/${LINEAGE}.json

# Add node names to internal nodes (necessary first step)
# Also generate timetree in augur (maybe this helps the mugration model??)
augur refine \
--tree $TREE \
--root "EPI_ISL_402125|2019-12-26" \
--keep-polytomies \
--output-tree $BASEDIR/$WORKDIR/output/augur/${LINEAGE}.nwk
# --timetree \
# --alignment $BASEDIR/$WORKDIR/tmp/alignments/${LINEAGE}.fasta \
# --metadata $METADATA \
# --clock-rate 0.0008 \

# Generate location reconstruction 
augur traits \
--tree $BASEDIR/$WORKDIR/output/augur/${LINEAGE}.nwk \
--metadata $METADATA \
--columns iso_country \
--confidence \
--output-node-data $OUTPUT_NODE_DATA

# Export file for tree visualization with Auspice
# augur export v2 \
# --tree $BASEDIR/$WORKDIR/output/augur/${LINEAGE}.nwk \
# --node-data $OUTPUT_NODE_DATA \
# --output $OUTPUT_AUSPICE
```

I can't get the timetree to export for auspice correctly
So I'll just spot-check the results here in R.
```{R}
augur_traits_b117_json <- jsonlite::read_json(
  path = paste(workdir, "output/augur", "B.1.1.7_augur_traits_iso_country.json", sep = "/"))
tree <- ape::read.tree(file = paste(workdir, "output/augur", "B.1.1.7.nwk", sep = "/"))

get_iso_country <- function(node) {
  return(node[["iso_country"]])
}
get_iso_country_confidence <- function(node) {
  iso_country_options <- node[["iso_country_confidence"]]
  iso_country_options_collapsed <- paste0(iso_country_options, collapse = ",")
  return(iso_country_options_collapsed)
}
get_iso_country_options <- function(node) {
  iso_country_options <- node[["iso_country_confidence"]]
  iso_country_options_collapsed <- paste0(names(iso_country_options), collapse = ",")
  return(iso_country_options_collapsed)
}

temp <- data.frame(
  node = names(augur_traits_b117_json$nodes),
  iso_country = unlist(lapply(X = augur_traits_b117_json$nodes, FUN = get_iso_country)),
  iso_country_confidence = unlist(lapply(X = augur_traits_b117_json$nodes, FUN = get_iso_country_confidence)),
  iso_country_options = unlist(lapply(X = augur_traits_b117_json$nodes, FUN = get_iso_country_options))
)
temp$node <- as.integer(temp$node)

require(ggtree)
ggtree(tr = tree) %<+% temp + 
  geom_nodelab(aes(label = iso_country_options)) + 
  geom_tiplab(aes(label = iso_country_options))
```

# Compare with parsimony result
```{r}
#' Plot the time-scaled phylogeny with tip locations, estimated ancestral node
#' locations, estimated Swiss transmission chains, and genetic similarity & 
#' travel context tips.
#' @param tree Grapevine output, e.g. /Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/jan-dec_-01_max_sampling_-5_context-sf/tmp/lsd/B.1.509.timetree.nex
#' @param tree_data_with_asr Grapevine output, e.g. /Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/jan-dec_-01_max_sampling_-5_context-sf/tmp/asr/B.1.509_m_3_p_1_s_F_tree_data_with_asr.txt
#' @param chains Grapevine output, e.g. /Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/jan-dec_-01_max_sampling_-5_context-sf/tmp/chains/B.1.509_m_3_p_1_s_F_chains.txt
plot_tree <- function(
  tree, tree_data_with_asr, chains, country_colors, outdir = NULL, prefix = "tree"
) {
  
  country_colors[["CHE"]] <- "red"
  country_colors_for_pies <- country_colors
  names(country_colors_for_pies) <- paste(names(country_colors), "_loc_weight", sep = "")
  
  tree_data_to_plot <- tree_data_with_asr %>% 
    mutate(
      node_annotation = "",
      # node_annotation = case_when(
      #   node %in% chains$ch_mrca ~ "Swiss MRCA",
      #   T ~ as.character(node)),
      tip_type  = case_when(
        travel_context ~ "travel context",
        similarity_context ~ "genetic context",
        focal_sequence ~ "",
        T ~ "root"))
  
  # Apply manual correction: LSD returns '2021' for date of tips with date '2020-12-31'
  tree_data_to_plot$date <- as.character(tree_data_to_plot$date)
  tree_data_to_plot[tree_data_to_plot$date == "2021" & !is.na(tree_data_to_plot$label) & grepl(x = tree_data_to_plot$label, pattern = "2020-12-31"), "date"] <- "2020-12-31"
  
  tree_to_plot <- full_join(tree, tree_data_to_plot)
  
  loc_weight_cols <- which(grepl(x = colnames(tree_data_to_plot), pattern = "_loc_weight"))
  node_location_pies <- ggtree::nodepie(
    tree_data_to_plot, 
    cols = loc_weight_cols)
  node_location_pies_colored <- lapply(
    X = node_location_pies, 
    FUN = function(g) g + scale_fill_manual(values = country_colors_for_pies))
  
  p <- ggtree(
    tr = tree_to_plot,
    mrsd = max(as.Date(tree_data_to_plot$date)), 
    as.Date = T) +
    geom_inset(insets = node_location_pies_colored) +
    geom_nodelab(aes(
      label = node_annotation)) +
    geom_tiplab(aes(
      label = iso_code_to_country_name(iso_country),
      # label = case_when(
      #   tip_type != "" ~ paste(iso_code_to_country_name(iso_country), tip_type, sep = ": "),
      #   iso_country == 'CHE' & iso_country_exposure != 'CHE' ~ paste(
      #     iso_code_to_country_name(iso_country), " (", 
      #     iso_code_to_country_name(iso_country_exposure), ")", sep = ""),
      #   T ~ iso_code_to_country_name(iso_country)),
      color = iso_country),
      hjust = -0.1) +
    scale_color_manual(values = country_colors) +
    scale_x_date(
      date_labels = "%b. %Y",
      limits = c(as.Date("2019-12-01"), as.Date("2021-06-01"))) + 
    theme_tree2() + 
    theme(legend.position = "none")
  show(p)
    
    
  if (is.null(outdir)) {
    return(p)
  } else {
    ggsave(
      file = paste(outdir, paste(prefix, "_with_asr.pdf", sep = ""), sep = "/"),
      plot = p,
      dpi = 400
    )
  }
}

tree <- ape::read.nexus(file = paste(workdir, "tmp/lsd", "B.1.1.7.timetree.nex", sep = "/"))
tree_data_with_asr <- read.delim(file = paste(workdir, "tmp/asr", "B.1.1.7_m_3_p_1_s_F_tree_data_with_asr.txt", sep = "/"))
chains <- read.delim(file = paste(workdir, "tmp/chains", "B.1.1.7_m_3_p_1_s_F_chains.txt", sep = "/"))
country_colors <- get_country_colors(db_connection)

ggtree(tr = tree)

plot_tree(
  tree = tree, 
  tree_data_with_asr = tree_data_with_asr, 
  chains = chains,
  country_colors = country_colors)
```