---
title: "Validate ancestral location inferece"
author: "Sarah Nadeau"
date: "31/03/2021"
output: html_document
---

```{r, setup, include=False}
knitr::opts_knit$set(root.dir = "/Users/nadeaus/Repos/grapevine")
knitr::opts_chunk$set(root.dir = "/Users/nadeaus/Repos/grapevine")
require(dplyr)
require(ggplot2)
require(ggtree)
```

Find pangolin lineages with Swiss sequences with recorded exposure information
```{postgresql}
select pangolin_lineage, iso_country_exp, count(*)
from bag_meldeformular bm
right join viollier_test vt on bm.sample_number = vt.sample_number
right join consensus_sequence cs on vt.ethid = cs.ethid
right join sequence_identifier si on vt.ethid = si.ethid
right join gisaid_sequence gs on gs.gisaid_epi_isl = si.gisaid_id
where iso_country_exp not in ('CHE', 'XXX') and
      pangolin_lineage in ('B.1.389', 'B.1.1.162')
group by pangolin_lineage, iso_country_exp;
```
pangolin_lineage,iso_country_exp,count
B.1.1.162,ALB,1
B.1.1.162,MKD,4
B.1.1.162,TUR,1
B.1.389,ITA,3
B.1.389,MLT,8

Which lineages are these?
```{postgresql}
select pangolin_lineage, iso_country_exp, gisaid_epi_isl, cs.ethid, order_date
from bag_meldeformular bm
right join viollier_test vt on bm.sample_number = vt.sample_number
right join consensus_sequence cs on vt.ethid = cs.ethid
right join sequence_identifier si on vt.ethid = si.ethid
right join gisaid_sequence gs on gs.gisaid_epi_isl = si.gisaid_id
where iso_country_exp not in ('CHE', 'XXX') and
      pangolin_lineage in ('B.1.389', 'B.1.1.162')
order by pangolin_lineage, iso_country_exp, order_date;
```
pangolin_lineage,iso_country_exp,gisaid_epi_isl,ethid,order_date
B.1.1.162,ALB,EPI_ISL_535612,250096,2020-08-19
B.1.1.162,MKD,EPI_ISL_489979,180041,2020-07-02
B.1.1.162,MKD,EPI_ISL_489980,180042,2020-07-02
B.1.1.162,MKD,EPI_ISL_1130781,210005,2020-07-17
B.1.1.162,MKD,EPI_ISL_523861,240006,2020-08-07
B.1.1.162,TUR,EPI_ISL_539342,260004,2020-08-21
B.1.389,ITA,EPI_ISL_603620,321374,2020-10-13
B.1.389,ITA,EPI_ISL_603682,321468,2020-10-14
B.1.389,ITA,EPI_ISL_603732,321523,2020-10-15
B.1.389,MLT,EPI_ISL_516598,230062,2020-08-05
B.1.389,MLT,EPI_ISL_523908,240070,2020-08-08
B.1.389,MLT,EPI_ISL_523876,240028,2020-08-10
B.1.389,MLT,EPI_ISL_523915,240077,2020-08-10
B.1.389,MLT,EPI_ISL_523875,240025,2020-08-10
B.1.389,MLT,EPI_ISL_523916,240078,2020-08-10
B.1.389,MLT,EPI_ISL_539378,260051,2020-08-24
B.1.389,MLT,EPI_ISL_560553,280209,2020-09-10

Get the geographic distribution of those lineages
```{postgresql}
select iso_country, count(*)
from gisaid_sequence
where pangolin_lineage = 'B.1.389'
group by iso_country
order by count(*) desc;

select iso_country, count(*)
from gisaid_sequence
where pangolin_lineage = 'B.1.1.162'
group by iso_country
order by count(*) desc;
```
iso_country,count
GBR,777
DNK,75
DEU,31
CHE,21
BEL,19
IRL,16
ITA,16 ***
MLT,10 ***
NOR,10
FRA,8
POL,7
ISL,5
LUX,4
NLD,3
SWE,2
CAN,2
ESP,2
AUT,2
RWA,1
CZE,1

iso_country,count
JPN,2663
GBR,442
USA,304
CAN,283
ESP,113
DEU,100
CHE,89
IND,70
ITA,60
MKD,57 ***
FRA,50
LVA,46
AUS,43
DNK,39
TUR,32 ***
ISR,26
IRL,25
ZMB,14
NLD,12
BRA,11
GRC,11
BEL,11
IDN,10
NOR,10
PRT,9
RUS,9
LUX,9
ISL,9
HUN,8
ARE,7
CZE,6
KOR,6
MEX,5
FIN,5
CHL,4
NZL,4
SWE,4
SGP,3
ARG,3
EGY,3
HKG,3
SVN,3
NGA,2
CHN,2
SEN,2
BWA,2
OMN,2
BLZ,2
THA,2
PER,2
URY,2
BGR,2
GEO,2
MAR,2
PSE,1
GMB,1
MOZ,1
MYS,1
EST,1
NPL,1
COL,1
COD,1
JOR,1
AUT,1
BHR,1
VNM,1

See what Nextstrain assigns these lineages

```{r}
swiss_nextstrain_metadata <- read.delim(
  file = "/Users/nadeaus/Downloads/nextstrain_groups_swiss_ncov_switzerland_metadata.tsv")
strains_sql <- "select pangolin_lineage, iso_country_exp, gisaid_epi_isl, cs.ethid, strain, order_date
from bag_meldeformular bm
right join viollier_test vt on bm.sample_number = vt.sample_number
right join consensus_sequence cs on vt.ethid = cs.ethid
right join sequence_identifier si on vt.ethid = si.ethid
right join gisaid_sequence gs on gs.gisaid_epi_isl = si.gisaid_id
where iso_country_exp not in ('CHE', 'XXX') and
      pangolin_lineage in ('B.1.389', 'B.1.1.162');"
db_connection <- open_database_connection("server")

res <- DBI::dbSendQuery(conn = db_connection, statement = strains_sql)
strains <- DBI::dbFetch(res = res)
DBI::dbClearResult(res = res)

strains_on_swiss_nextstrain <- merge(x = swiss_nextstrain_metadata, y = strains,
                                     by.x = "Strain", by.y = "strain")
```
These tips are in the tree:
Switzerland/SG-ETHZ-240006/2020 (MKD)
Switzerland/SG-ETHZ-240025/2020 (MLT)
Switzerland/SG-ETHZ-240028/2020 (MLT)
Switzerland/UR-ETHZ-240077/2020 (MLT)

The ancestral locations on the whole nextstrain tree beyond the very top nodes
(Asia) are Swiss, essentially.

What do I get for the ASRs of these tips?
```{bash}
WORKDIR=/Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/20210330/jan-dec_-01_max_sampling_1_travel_-5_sim_context-sf_1_exp-wt
TMP_ALIGNMENTS=$WORKDIR/tmp/alignments
TMP_CHAINS=$WORKDIR/tmp/chains
TMP_LSD=$WORKDIR/tmp/lsd
TMP_ASR=$WORKDIR/post_run_tmp/asr
mkdir -p $TMP_ASR

MAX_NONFOCAL_SUBCLADES=3
MAX_CONSECUTIVE_BUDDING_NONFOCAL_SUBCLADES=1

for TREEFILE in $TMP_LSD/B.1.1.162.timetree.nex ; do
    PREFIX="$(basename "${TREEFILE}" | sed 's/.timetree.nex//g')"
    Rscript analyze_tree/reconstruct_ancestral_locations_weighted_parsimony.R \
        --tree $TREEFILE \
        --metadata $TMP_ALIGNMENTS/${PREFIX}_metadata.csv \
        --chains $TMP_CHAINS/${PREFIX}_m_${MAX_NONFOCAL_SUBCLADES}_p_${MAX_CONSECUTIVE_BUDDING_NONFOCAL_SUBCLADES}_s_T_chains.txt \
        --outdir $TMP_ASR \
        --prefix ${PREFIX}_m_${MAX_NONFOCAL_SUBCLADES}_p_${MAX_CONSECUTIVE_BUDDING_NONFOCAL_SUBCLADES}_s_T \
        --plottree 
done

for TREEFILE in $TMP_LSD/B.1.1.162.timetree.nex ; do 
    PREFIX="$(basename "${TREEFILE}" | sed 's/.timetree.nex//g')"
    Rscript analyze_tree/reconstruct_ancestral_locations_weighted_parsimony.R \
        --tree $TREEFILE \
        --metadata $TMP_ALIGNMENTS/${PREFIX}_metadata.csv \
        --chains $TMP_CHAINS/${PREFIX}_m_${MAX_NONFOCAL_SUBCLADES}_p_${MAX_CONSECUTIVE_BUDDING_NONFOCAL_SUBCLADES}_s_F_chains.txt \
        --outdir $TMP_ASR \
        --prefix ${PREFIX}_m_${MAX_NONFOCAL_SUBCLADES}_p_${MAX_CONSECUTIVE_BUDDING_NONFOCAL_SUBCLADES}_s_F \
        --plottree 
done
```
Many samples won't have been included because, e.g.: 
"sampling 6 Swiss samples from 47 in week 2020-06-29"

In the B.1.1.162 tree the MKD sequence 210005 is a descendent of a polytomy with prob:
~1/6 MKD
~1/6 TUR
~2/3 FRA
~1/6 DEU
~1/6 BEL & CAN

What does the USCS genome browser say?
I uploaded B.1.1.162.fasta to https://genome.ucsc.edu/cgi-bin/hgPhyloPlace and chose 250,000 genomes from GISAID; audacity tree.
I get that my MKD sequence 210005 has nearest neighbor in Norway, is in clade defined by mutation C22205G with:
2 x CHE
2 x MKD
1 x NOR
These sequences are:
Switzerland/BS-42346247/2020|EPI_ISL_581883|20-07-09
Switzerland/BS-42343508/2020|EPI_ISL_581880|20-07-07
NorthMacedonia/ChVir7914/2020|EPI_ISL_492084|20-06-25
NorthMacedonia/ChVir7908/2020|EPI_ISL_492079|20-06-26
Norway/3007/2020|EPI_ISL_549071|20-07-31

Intriguingly, all of these sequences are labelled as B.1.1 in the genome browser, whereas I have 
MKD sequence 210005 in the B.1.1.162 tree. Are the PANGO lineages really misleading?

Is my collapse short branches criterion too strict, or did I just not include any other C22205G sequences in my tree?
```{r}
B11162_metadata <- read.csv(
  file = "/Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/20210330/jan-dec_-01_max_sampling_1_travel_-5_sim_context-sf_1_exp-wt/tmp/alignments/B.1.1.162_metadata.csv")
nearest_epi_isls <- c("EPI_ISL_581883", "EPI_ISL_581880", "EPI_ISL_492084", "EPI_ISL_492079", "EPI_ISL_549071")

B11162_metadata %>% filter(
  gisaid_epi_isl %in% nearest_epi_isls)
```
I didn't include any other C22205G sequences in my tree, so it's fair that 
MKD sequence 210005 descends from a polytomy in my tree.

## Conclusions
Based on a spot-check of _one_ samples with known travel exposure, my method is
slightly worse than the state-of-the-art genome browser, and over-weights neighboring
countries of Switzerland. 

So the differences between the prior and the posterior may be informative. 
But how can I validate better?

Ideas:
Precision: check how much the deviations from the prior differ with different draws of context sequences
Accuracy: check how much deviation from the prior for the known exposure seqs is for the correct countries

Copy results to local
```{bash}
WORKDIR=jan-dec_-005_max_sampling_-25_travel_-5_sim_context-sf_111_travel-wt
mkdir -p /Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/validation/$WORKDIR

scp nadeaus@euler.ethz.ch:/cluster/scratch/nadeaus/grapevine/$WORKDIR/*.log \
/Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/validation/$WORKDIR

rsync -a --relative \
nadeaus@euler.ethz.ch:/cluster/scratch/nadeaus/grapevine/$WORKDIR/./output/* \
/Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/validation/$WORKDIR

rsync -a --relative \
nadeaus@euler.ethz.ch:/cluster/scratch/nadeaus/grapevine/$WORKDIR/./tmp/*/*.csv \
/Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/validation/$WORKDIR

rsync -a --relative \
nadeaus@euler.ethz.ch:/cluster/scratch/nadeaus/grapevine/$WORKDIR/./tmp/*/*.txt \
/Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/validation/$WORKDIR

rsync -a --relative \
nadeaus@euler.ethz.ch:/cluster/scratch/nadeaus/grapevine/$WORKDIR/./tmp/*/*.nex \
/Users/nadeaus/Repos/cov-swiss-phylogenetics/results_all/validation/$WORKDIR
```

Load results
```{r}
source("generate_figures/functions.R")

workdir <- paste(
  "../cov-swiss-phylogenetics/results_all/validation",
  "jan-dec_-005_max_sampling_-25_travel_-5_sim_context-sf_111_travel-wt",
  sep = "/")

grapevine_results <- load_grapevine_results(workdir = workdir)
```

Calculate accuracy
```{r}
validation_samples <- grapevine_results$samples %>%
  filter(!(iso_country_exposure %in% c('CHE', 'XXX')))

validation_samples %>%
  select(sample, iso_country_exposure, tree)
```
Only 6 samples included with recorded exposure information, all in the same tree
and all from either France or Germany

```{r}
validation_samples_chains <- merge(
  x = validation_samples,
  y = grapevine_results$chains,
  by = c("tree", "chains_assumption", "chain_idx"),
  all.x = T)

validation_samples_origins <- merge(
  x = validation_samples_chains,
  y = grapevine_results$origins,
  by = c("tree", "chains_assumption", "foreign_mrca"),
  all.x = T)

validation_samples_origins_long <- pivot_origins_longer(origins = validation_samples_origins) %>%
  filter(!is.na(asr_contribution), asr_contribution > 0)

validation_samples_origins_long %>% 
  select("chains_assumption", "sample", "origin", "asr_contribution", "iso_country_exposure", "foreign_mrca") %>%
  group_by(chains_assumption, sample) %>%
  arrange(desc(asr_contribution), .by_group = T)
```
3 samples have ASR information.
Under the min chains assumption they all descend from the same Swiss polytomy.
Under the max chains assumption, the 2x French and 1x German all descend from the same foreign MRCA that is:
52% French, 22% Italian, 22% German, 4% UK.

This indicates the max chains assumption is more realistic and 
that in this _one_ case, the proportional ASR is fairly representative of where chains came from.

However, 0.5% per week Swiss sampling doesn't by chance include enough seqs with
travel exposure information to do a proper accuracy validation analysis.

Accuracy take 2: include only Swiss sequences with travel exposure information for phylogenetic analysis.


